<!DOCTYPE html>
<!-- saved from url=(0077)http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub -->
<html lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><iframe src="javascript:false" title="" style="width: 0px; height: 0px; border: 0px; display: none;" src="./Developing a corpus of clinical notes manually annotated for part-of-speech - ScienceDirect_files/saved_resource.html"></iframe><script async="" src="https://connect.facebook.net/en_US/fbevents.js"></script><script type="text/javascript" async="" src="./Developing a corpus of clinical notes manually annotated for part-of-speech - ScienceDirect_files/geo2.js"></script>
      <meta name="citation_pii" content="S1386505605001723">
<meta name="citation_issn" content="1386-5056">
<meta name="citation_volume" content="75">
<meta name="citation_lastpage" content="429">
<meta name="citation_issue" content="6">
<meta name="citation_publisher" content="Elsevier">
<meta name="citation_firstpage" content="418">
<meta name="citation_journal_title" content="International Journal of Medical Informatics">
<meta name="citation_type" content="JOUR">
<meta name="citation_doi" content="10.1016/j.ijmedinf.2005.08.006">
<meta name="dc.identifier" content="10.1016/j.ijmedinf.2005.08.006">
<meta name="citation_article_type" content="Full-length article">
<meta name="citation_title" content="Developing a corpus of clinical notes manually annotated for part-of-speech">
<meta name="citation_publication_date" content="2006/06/01">
<meta name="citation_online_date" content="2005/09/19">
<meta name="citation_pdf_url" content="https://www.sciencedirect.com/science/article/pii/S1386505605001723/pdfft?md5=566710269dc6c26937e3e391a2dd14ff&amp;pid=1-s2.0-S1386505605001723-main.pdf">
<meta name="robots" content="INDEX,FOLLOW,NOARCHIVE,NOODP,NOYDIR">
      <title>Developing a corpus of clinical notes manually annotated for part-of-speech - ScienceDirect</title>
      <link rel="canonical" href="https://www.sciencedirect.com/science/article/pii/S1386505605001723">
      <meta name="viewport" content="initial-scale=1">
      <meta name="SDTech" content="Proudly brought to you by the SD Technology team in London, Dayton, and Amsterdam">
      <link rel="shortcut icon" href="https://cdn.els-cdn.com/sd/favSD.ico" type="image/x-icon">
      <link rel="icon" href="https://cdn.els-cdn.com/sd/favSD.ico" type="image/x-icon">
      <link rel="stylesheet" href="./Developing a corpus of clinical notes manually annotated for part-of-speech - ScienceDirect_files/arp.css">
      <link rel="stylesheet" href="./Developing a corpus of clinical notes manually annotated for part-of-speech - ScienceDirect_files/style.css">
      <link rel="dns-prefetch" href="https://w.usabilla.com/">
      <link rel="dns-prefetch" href="https://www.deepdyve.com/">
      <link rel="dns-prefetch" href="https://smetrics.elsevier.com/">
      <script>
        window.pageTargeting = {"region":"us-west-2","platform":"sdtech","entitled":true};
        window.pageData = {"content":[{"id":"sd:article:pii:s1386505605001723","entitlementType":"package","format":"MIME-XHTML","type":"SD:ARTICLE:SCOPE-FULL","detail":"SD:ARTICLE:subtype:fla"}],"page":{"businessUnit":"ELS:RP:ST","language":"en","name":"product:journal:article","noTracking":"false","productName":"SD","type":"CP-CA","environment":"prod","loadTimestamp":1515719499412,"loadTime":""},"visitor":{"accessType":"ae:ANON_IP","accountId":"ae:50229","accountName":"ae:University of Victoria","loginStatus":"anonymous","userId":"ae:1007916","ipAddress":"206.87.163.83"}};
        window.arp = {
          subscriptions: [],
          subscribe: function(cb) {
            var self = this;
            var i = this.subscriptions.push(cb) - 1;
            return function unsubscribe() {
              self.subscriptions.splice(i, 1);
            }
          },
        };
      </script>
      <script type="text/javascript" src="./Developing a corpus of clinical notes manually annotated for part-of-speech - ScienceDirect_files/204774041.js"></script>
      <!-- begin mPulse embed code -->
      <script>
        (function(){
          if(window.BOOMR && window.BOOMR.version){return;}
          var dom,doc,where,iframe = document.createElement('iframe'),win = window;

          function boomerangSaveLoadTime(e) {
            win.BOOMR_onload=(e && e.timeStamp) || new Date().getTime();
          }
          if (win.addEventListener) {
            win.addEventListener("load", boomerangSaveLoadTime, false);
          } else if (win.attachEvent) {
            win.attachEvent("onload", boomerangSaveLoadTime);
          }

          iframe.src = "javascript:false";
          iframe.title = ""; iframe.role="presentation";
          (iframe.frameElement || iframe).style.cssText = "width:0;height:0;border:0;display:none;";
          where = document.getElementsByTagName('script')[0];
          where.parentNode.insertBefore(iframe, where);

          try {
            doc = iframe.contentWindow.document;
          } catch(e) {
            dom = document.domain;
            iframe.src="javascript:var d=document.open();d.domain='"+dom+"';void(0);";
            doc = iframe.contentWindow.document;
          }
          doc.open()._l = function() {
            var js = this.createElement("script");
            if(dom) this.domain = dom;
            js.id = "boomr-if-as";
            js.src = 'https://c.go-mpulse.net/boomerang/2FBN2-NKMGU-EJKY8-ZANKZ-SUJZF';
            BOOMR_lstart=new Date().getTime();
            this.body.appendChild(js);
          };
          doc.write('<body onload="document._l();">');
          doc.close();
        })();
      </script>
      <!-- end mPulse embed code -->
    <script src="https://googleads.g.doubleclick.net/pagead/viewthroughconversion/1017027555/?random=1515719500225&amp;cv=8&amp;fst=1515719500225&amp;num=1&amp;guid=ON&amp;eid=659238991&amp;u_h=768&amp;u_w=1366&amp;u_ah=683&amp;u_aw=1366&amp;u_cd=24&amp;u_his=1&amp;u_tz=-480&amp;u_java=false&amp;u_nplug=3&amp;u_nmime=3&amp;frm=0&amp;url=http%3A%2F%2Fwww.sciencedirect.com%2Fscience%2Farticle%2Fpii%2FS1386505605001723%3Fvia%253Dihub&amp;ref=http%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS1386-5056(05)00172-3&amp;tiba=Developing%20a%20corpus%20of%20clinical%20notes%20manually%20annotated%20for%20part-of-speech%20-%20ScienceDirect&amp;rfmt=3&amp;fmt=4"></script><script src="./Developing a corpus of clinical notes manually annotated for part-of-speech - ScienceDirect_files/satellite-5964b08664746d3292014dba.js"></script><script src="./Developing a corpus of clinical notes manually annotated for part-of-speech - ScienceDirect_files/s-code-contents-9c0358adbc3b5986e210099b3bf1d427fc5bd286.js"></script><link rel="preload" href="https://adservice.google.ca/adsid/integrator.js?domain=www.sciencedirect.com" as="script"><script type="text/javascript" src="https://adservice.google.ca/adsid/integrator.js?domain=www.sciencedirect.com"></script><link rel="preload" href="https://adservice.google.com/adsid/integrator.js?domain=www.sciencedirect.com" as="script"><script type="text/javascript" src="https://adservice.google.com/adsid/integrator.js?domain=www.sciencedirect.com"></script><script src="https://securepubads.g.doubleclick.net/gpt/pubads_impl_172.js" async=""></script><link id="plx-css-summary" type="text/css" rel="stylesheet" href="./Developing a corpus of clinical notes manually annotated for part-of-speech - ScienceDirect_files/summary.css"><script type="text/javascript" src="./Developing a corpus of clinical notes manually annotated for part-of-speech - ScienceDirect_files/jquery.min.js"></script><script type="text/javascript" src="./Developing a corpus of clinical notes manually annotated for part-of-speech - ScienceDirect_files/xss.js"></script><link rel="prefetch" href="http://tpc.googlesyndication.com/safeframe/1-0-14/html/container.html"><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style><style type="text/css">.MJX_Assistive_MathML {position: absolute!important; top: 0; left: 0; clip: rect(1px, 1px, 1px, 1px); padding: 1px 0 0 0!important; border: 0!important; height: 1px!important; width: 1px!important; overflow: hidden!important; display: block!important; -webkit-touch-callout: none; -webkit-user-select: none; -khtml-user-select: none; -moz-user-select: none; -ms-user-select: none; user-select: none}
.MJX_Assistive_MathML.MJX_Assistive_MathML_Block {width: 100%!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1em; bottom: 1.5em; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MJXp-script {font-size: .8em}
.MJXp-right {-webkit-transform-origin: right; -moz-transform-origin: right; -ms-transform-origin: right; -o-transform-origin: right; transform-origin: right}
.MJXp-bold {font-weight: bold}
.MJXp-italic {font-style: italic}
.MJXp-scr {font-family: MathJax_Script,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-frak {font-family: MathJax_Fraktur,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-sf {font-family: MathJax_SansSerif,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-cal {font-family: MathJax_Caligraphic,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-mono {font-family: MathJax_Typewriter,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-largeop {font-size: 150%}
.MJXp-largeop.MJXp-int {vertical-align: -.2em}
.MJXp-math {display: inline-block; line-height: 1.2; text-indent: 0; font-family: 'Times New Roman',Times,STIXGeneral,serif; white-space: nowrap; border-collapse: collapse}
.MJXp-display {display: block; text-align: center; margin: 1em 0}
.MJXp-math span {display: inline-block}
.MJXp-box {display: block!important; text-align: center}
.MJXp-box:after {content: " "}
.MJXp-rule {display: block!important; margin-top: .1em}
.MJXp-char {display: block!important}
.MJXp-mo {margin: 0 .15em}
.MJXp-mfrac {margin: 0 .125em; vertical-align: .25em}
.MJXp-denom {display: inline-table!important; width: 100%}
.MJXp-denom > * {display: table-row!important}
.MJXp-surd {vertical-align: top}
.MJXp-surd > * {display: block!important}
.MJXp-script-box > *  {display: table!important; height: 50%}
.MJXp-script-box > * > * {display: table-cell!important; vertical-align: top}
.MJXp-script-box > *:last-child > * {vertical-align: bottom}
.MJXp-script-box > * > * > * {display: block!important}
.MJXp-mphantom {visibility: hidden}
.MJXp-munderover {display: inline-table!important}
.MJXp-over {display: inline-block!important; text-align: center}
.MJXp-over > * {display: block!important}
.MJXp-munderover > * {display: table-row!important}
.MJXp-mtable {vertical-align: .25em; margin: 0 .125em}
.MJXp-mtable > * {display: inline-table!important; vertical-align: middle}
.MJXp-mtr {display: table-row!important}
.MJXp-mtd {display: table-cell!important; text-align: center; padding: .5em 0 0 .5em}
.MJXp-mtr > .MJXp-mtd:first-child {padding-left: 0}
.MJXp-mtr:first-child > .MJXp-mtd {padding-top: 0}
.MJXp-mlabeledtr {display: table-row!important}
.MJXp-mlabeledtr > .MJXp-mtd:first-child {padding-left: 0}
.MJXp-mlabeledtr:first-child > .MJXp-mtd {padding-top: 0}
.MJXp-merror {background-color: #FFFF88; color: #CC0000; border: 1px solid #CC0000; padding: 1px 3px; font-style: normal; font-size: 90%}
.MJXp-scale0 {-webkit-transform: scaleX(.0); -moz-transform: scaleX(.0); -ms-transform: scaleX(.0); -o-transform: scaleX(.0); transform: scaleX(.0)}
.MJXp-scale1 {-webkit-transform: scaleX(.1); -moz-transform: scaleX(.1); -ms-transform: scaleX(.1); -o-transform: scaleX(.1); transform: scaleX(.1)}
.MJXp-scale2 {-webkit-transform: scaleX(.2); -moz-transform: scaleX(.2); -ms-transform: scaleX(.2); -o-transform: scaleX(.2); transform: scaleX(.2)}
.MJXp-scale3 {-webkit-transform: scaleX(.3); -moz-transform: scaleX(.3); -ms-transform: scaleX(.3); -o-transform: scaleX(.3); transform: scaleX(.3)}
.MJXp-scale4 {-webkit-transform: scaleX(.4); -moz-transform: scaleX(.4); -ms-transform: scaleX(.4); -o-transform: scaleX(.4); transform: scaleX(.4)}
.MJXp-scale5 {-webkit-transform: scaleX(.5); -moz-transform: scaleX(.5); -ms-transform: scaleX(.5); -o-transform: scaleX(.5); transform: scaleX(.5)}
.MJXp-scale6 {-webkit-transform: scaleX(.6); -moz-transform: scaleX(.6); -ms-transform: scaleX(.6); -o-transform: scaleX(.6); transform: scaleX(.6)}
.MJXp-scale7 {-webkit-transform: scaleX(.7); -moz-transform: scaleX(.7); -ms-transform: scaleX(.7); -o-transform: scaleX(.7); transform: scaleX(.7)}
.MJXp-scale8 {-webkit-transform: scaleX(.8); -moz-transform: scaleX(.8); -ms-transform: scaleX(.8); -o-transform: scaleX(.8); transform: scaleX(.8)}
.MJXp-scale9 {-webkit-transform: scaleX(.9); -moz-transform: scaleX(.9); -ms-transform: scaleX(.9); -o-transform: scaleX(.9); transform: scaleX(.9)}
</style><style type="text/css" media="print">.usabilla_live_button_container { display: none; }</style><style type="text/css">.MathJax_SVG_Display {text-align: center; margin: 1em 0em; position: relative; display: block!important; text-indent: 0; max-width: none; max-height: none; min-width: 0; min-height: 0; width: 100%}
.MathJax_SVG .MJX-monospace {font-family: monospace}
.MathJax_SVG .MJX-sans-serif {font-family: sans-serif}
#MathJax_SVG_Tooltip {background-color: InfoBackground; color: InfoText; border: 1px solid black; box-shadow: 2px 2px 5px #AAAAAA; -webkit-box-shadow: 2px 2px 5px #AAAAAA; -moz-box-shadow: 2px 2px 5px #AAAAAA; -khtml-box-shadow: 2px 2px 5px #AAAAAA; padding: 3px 4px; z-index: 401; position: absolute; left: 0; top: 0; width: auto; height: auto; display: none}
.MathJax_SVG {display: inline; font-style: normal; font-weight: normal; line-height: normal; font-size: 100%; font-size-adjust: none; text-indent: 0; text-align: left; text-transform: none; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; direction: ltr; max-width: none; max-height: none; min-width: 0; min-height: 0; border: 0; padding: 0; margin: 0}
.MathJax_SVG * {transition: none; -webkit-transition: none; -moz-transition: none; -ms-transition: none; -o-transition: none}
.mjx-svg-href {fill: blue; stroke: blue}
.MathJax_SVG_Processing {visibility: hidden; position: absolute; top: 0; left: 0; width: 0; height: 0; overflow: hidden; display: block!important}
.MathJax_SVG_Processed {display: none!important}
.MathJax_SVG_ExBox {display: block!important; overflow: hidden; width: 1px; height: 60ex; min-height: 0; max-height: none; padding: 0; border: 0; margin: 0}
.MathJax_SVG_LineBox {display: table!important}
.MathJax_SVG_LineBox span {display: table-cell!important; width: 10000em!important; min-width: 0; max-width: none; padding: 0; border: 0; margin: 0}
</style></head>
    <body style=""><div style="visibility: hidden; overflow: hidden; position: absolute; top: 0px; height: 1px; width: auto; padding: 0px; border: 0px; margin: 0px; text-align: left; text-indent: 0px; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal;"><div id="MathJax_SVG_Hidden"></div><svg><defs id="MathJax_SVG_glyphs"><path stroke-width="1" id="MJMAIN-41" d="M255 0Q240 3 140 3Q48 3 39 0H32V46H47Q119 49 139 88Q140 91 192 245T295 553T348 708Q351 716 366 716H376Q396 715 400 709Q402 707 508 390L617 67Q624 54 636 51T687 46H717V0H708Q699 3 581 3Q458 3 437 0H427V46H440Q510 46 510 64Q510 66 486 138L462 209H229L209 150Q189 91 189 85Q189 72 209 59T259 46H264V0H255ZM447 255L345 557L244 256Q244 255 345 255H447Z"></path><path stroke-width="1" id="MJMAIN-62" d="M307 -11Q234 -11 168 55L158 37Q156 34 153 28T147 17T143 10L138 1L118 0H98V298Q98 599 97 603Q94 622 83 628T38 637H20V660Q20 683 22 683L32 684Q42 685 61 686T98 688Q115 689 135 690T165 693T176 694H179V543Q179 391 180 391L183 394Q186 397 192 401T207 411T228 421T254 431T286 439T323 442Q401 442 461 379T522 216Q522 115 458 52T307 -11ZM182 98Q182 97 187 90T196 79T206 67T218 55T233 44T250 35T271 29T295 26Q330 26 363 46T412 113Q424 148 424 212Q424 287 412 323Q385 405 300 405Q270 405 239 390T188 347L182 339V98Z"></path><path stroke-width="1" id="MJMAIN-73" d="M295 316Q295 356 268 385T190 414Q154 414 128 401Q98 382 98 349Q97 344 98 336T114 312T157 287Q175 282 201 278T245 269T277 256Q294 248 310 236T342 195T359 133Q359 71 321 31T198 -10H190Q138 -10 94 26L86 19L77 10Q71 4 65 -1L54 -11H46H42Q39 -11 33 -5V74V132Q33 153 35 157T45 162H54Q66 162 70 158T75 146T82 119T101 77Q136 26 198 26Q295 26 295 104Q295 133 277 151Q257 175 194 187T111 210Q75 227 54 256T33 318Q33 357 50 384T93 424T143 442T187 447H198Q238 447 268 432L283 424L292 431Q302 440 314 448H322H326Q329 448 335 442V310L329 304H301Q295 310 295 316Z"></path><path stroke-width="1" id="MJMAIN-61" d="M137 305T115 305T78 320T63 359Q63 394 97 421T218 448Q291 448 336 416T396 340Q401 326 401 309T402 194V124Q402 76 407 58T428 40Q443 40 448 56T453 109V145H493V106Q492 66 490 59Q481 29 455 12T400 -6T353 12T329 54V58L327 55Q325 52 322 49T314 40T302 29T287 17T269 6T247 -2T221 -8T190 -11Q130 -11 82 20T34 107Q34 128 41 147T68 188T116 225T194 253T304 268H318V290Q318 324 312 340Q290 411 215 411Q197 411 181 410T156 406T148 403Q170 388 170 359Q170 334 154 320ZM126 106Q126 75 150 51T209 26Q247 26 276 49T315 109Q317 116 318 175Q318 233 317 233Q309 233 296 232T251 223T193 203T147 166T126 106Z"></path><path stroke-width="1" id="MJMAIN-67" d="M329 409Q373 453 429 453Q459 453 472 434T485 396Q485 382 476 371T449 360Q416 360 412 390Q410 404 415 411Q415 412 416 414V415Q388 412 363 393Q355 388 355 386Q355 385 359 381T368 369T379 351T388 325T392 292Q392 230 343 187T222 143Q172 143 123 171Q112 153 112 133Q112 98 138 81Q147 75 155 75T227 73Q311 72 335 67Q396 58 431 26Q470 -13 470 -72Q470 -139 392 -175Q332 -206 250 -206Q167 -206 107 -175Q29 -140 29 -75Q29 -39 50 -15T92 18L103 24Q67 55 67 108Q67 155 96 193Q52 237 52 292Q52 355 102 398T223 442Q274 442 318 416L329 409ZM299 343Q294 371 273 387T221 404Q192 404 171 388T145 343Q142 326 142 292Q142 248 149 227T179 192Q196 182 222 182Q244 182 260 189T283 207T294 227T299 242Q302 258 302 292T299 343ZM403 -75Q403 -50 389 -34T348 -11T299 -2T245 0H218Q151 0 138 -6Q118 -15 107 -34T95 -74Q95 -84 101 -97T122 -127T170 -155T250 -167Q319 -167 361 -139T403 -75Z"></path><path stroke-width="1" id="MJMAIN-72" d="M36 46H50Q89 46 97 60V68Q97 77 97 91T98 122T98 161T98 203Q98 234 98 269T98 328L97 351Q94 370 83 376T38 385H20V408Q20 431 22 431L32 432Q42 433 60 434T96 436Q112 437 131 438T160 441T171 442H174V373Q213 441 271 441H277Q322 441 343 419T364 373Q364 352 351 337T313 322Q288 322 276 338T263 372Q263 381 265 388T270 400T273 405Q271 407 250 401Q234 393 226 386Q179 341 179 207V154Q179 141 179 127T179 101T180 81T180 66V61Q181 59 183 57T188 54T193 51T200 49T207 48T216 47T225 47T235 46T245 46H276V0H267Q249 3 140 3Q37 3 28 0H20V46H36Z"></path><path stroke-width="1" id="MJMAIN-3D" d="M56 347Q56 360 70 367H707Q722 359 722 347Q722 336 708 328L390 327H72Q56 332 56 347ZM56 153Q56 168 72 173H708Q722 163 722 153Q722 140 707 133H70Q56 140 56 153Z"></path><path stroke-width="1" id="MJMAIN-31" d="M213 578L200 573Q186 568 160 563T102 556H83V602H102Q149 604 189 617T245 641T273 663Q275 666 285 666Q294 666 302 660V361L303 61Q310 54 315 52T339 48T401 46H427V0H416Q395 3 257 3Q121 3 100 0H88V46H114Q136 46 152 46T177 47T193 50T201 52T207 57T213 61V578Z"></path><path stroke-width="1" id="MJMAIN-30" d="M96 585Q152 666 249 666Q297 666 345 640T423 548Q460 465 460 320Q460 165 417 83Q397 41 362 16T301 -15T250 -22Q224 -22 198 -16T137 16T82 83Q39 165 39 320Q39 494 96 585ZM321 597Q291 629 250 629Q208 629 178 597Q153 571 145 525T137 333Q137 175 145 125T181 46Q209 16 250 16Q290 16 318 46Q347 76 354 130T362 333Q362 478 354 524T321 597Z"></path><path stroke-width="1" id="MJMAIN-D7" d="M630 29Q630 9 609 9Q604 9 587 25T493 118L389 222L284 117Q178 13 175 11Q171 9 168 9Q160 9 154 15T147 29Q147 36 161 51T255 146L359 250L255 354Q174 435 161 449T147 471Q147 480 153 485T168 490Q173 490 175 489Q178 487 284 383L389 278L493 382Q570 459 587 475T609 491Q630 491 630 471Q630 464 620 453T522 355L418 250L522 145Q606 61 618 48T630 29Z"></path><path stroke-width="1" id="MJMAIN-28" d="M94 250Q94 319 104 381T127 488T164 576T202 643T244 695T277 729T302 750H315H319Q333 750 333 741Q333 738 316 720T275 667T226 581T184 443T167 250T184 58T225 -81T274 -167T316 -220T333 -241Q333 -250 318 -250H315H302L274 -226Q180 -141 137 -14T94 250Z"></path><path stroke-width="1" id="MJMATHI-4D" d="M289 629Q289 635 232 637Q208 637 201 638T194 648Q194 649 196 659Q197 662 198 666T199 671T201 676T203 679T207 681T212 683T220 683T232 684Q238 684 262 684T307 683Q386 683 398 683T414 678Q415 674 451 396L487 117L510 154Q534 190 574 254T662 394Q837 673 839 675Q840 676 842 678T846 681L852 683H948Q965 683 988 683T1017 684Q1051 684 1051 673Q1051 668 1048 656T1045 643Q1041 637 1008 637Q968 636 957 634T939 623Q936 618 867 340T797 59Q797 55 798 54T805 50T822 48T855 46H886Q892 37 892 35Q892 19 885 5Q880 0 869 0Q864 0 828 1T736 2Q675 2 644 2T609 1Q592 1 592 11Q592 13 594 25Q598 41 602 43T625 46Q652 46 685 49Q699 52 704 61Q706 65 742 207T813 490T848 631L654 322Q458 10 453 5Q451 4 449 3Q444 0 433 0Q418 0 415 7Q413 11 374 317L335 624L267 354Q200 88 200 79Q206 46 272 46H282Q288 41 289 37T286 19Q282 3 278 1Q274 0 267 0Q265 0 255 0T221 1T157 2Q127 2 95 1T58 0Q43 0 39 2T35 11Q35 13 38 25T43 40Q45 46 65 46Q135 46 154 86Q158 92 223 354T289 629Z"></path><path stroke-width="1" id="MJMATHI-54" d="M40 437Q21 437 21 445Q21 450 37 501T71 602L88 651Q93 669 101 677H569H659Q691 677 697 676T704 667Q704 661 687 553T668 444Q668 437 649 437Q640 437 637 437T631 442L629 445Q629 451 635 490T641 551Q641 586 628 604T573 629Q568 630 515 631Q469 631 457 630T439 622Q438 621 368 343T298 60Q298 48 386 46Q418 46 427 45T436 36Q436 31 433 22Q429 4 424 1L422 0Q419 0 415 0Q410 0 363 1T228 2Q99 2 64 0H49Q43 6 43 9T45 27Q49 40 55 46H83H94Q174 46 189 55Q190 56 191 56Q196 59 201 76T241 233Q258 301 269 344Q339 619 339 625Q339 630 310 630H279Q212 630 191 624Q146 614 121 583T67 467Q60 445 57 441T43 437H40Z"></path><path stroke-width="1" id="MJMAIN-29" d="M60 749L64 750Q69 750 74 750H86L114 726Q208 641 251 514T294 250Q294 182 284 119T261 12T224 -76T186 -143T145 -194T113 -227T90 -246Q87 -249 86 -250H74Q66 -250 63 -250T58 -247T55 -238Q56 -237 66 -225Q221 -64 221 250T66 725Q56 737 55 738Q55 746 60 749Z"></path><path stroke-width="1" id="MJSZ1-28" d="M152 251Q152 646 388 850H416Q422 844 422 841Q422 837 403 816T357 753T302 649T255 482T236 250Q236 124 255 19T301 -147T356 -251T403 -315T422 -340Q422 -343 416 -349H388Q359 -325 332 -296T271 -213T212 -97T170 56T152 251Z"></path><path stroke-width="1" id="MJSZ1-29" d="M305 251Q305 -145 69 -349H56Q43 -349 39 -347T35 -338Q37 -333 60 -307T108 -239T160 -136T204 27T221 250T204 473T160 636T108 740T60 807T35 839Q35 850 50 850H56H69Q197 743 256 566Q305 425 305 251Z"></path><path stroke-width="1" id="MJMAIN-4B" d="M128 622Q121 629 117 631T101 634T58 637H25V683H36Q57 680 180 680Q315 680 324 683H335V637H313Q235 637 233 620Q232 618 232 462L233 307L379 449Q425 494 479 546Q518 584 524 591T531 607V608Q531 630 503 636Q501 636 498 636T493 637H489V683H499Q517 680 630 680Q704 680 716 683H722V637H708Q633 633 589 597Q584 592 495 506T406 419T515 254T631 80Q644 60 662 54T715 46H736V0H728Q719 3 615 3Q493 3 472 0H461V46H469Q515 46 515 72Q515 78 512 84L336 351Q332 348 278 296L232 251V156Q232 62 235 58Q243 47 302 46H335V0H324Q303 3 180 3Q45 3 36 0H25V46H58Q100 47 109 49T128 61V622Z"></path><path stroke-width="1" id="MJMAIN-70" d="M36 -148H50Q89 -148 97 -134V-126Q97 -119 97 -107T97 -77T98 -38T98 6T98 55T98 106Q98 140 98 177T98 243T98 296T97 335T97 351Q94 370 83 376T38 385H20V408Q20 431 22 431L32 432Q42 433 61 434T98 436Q115 437 135 438T165 441T176 442H179V416L180 390L188 397Q247 441 326 441Q407 441 464 377T522 216Q522 115 457 52T310 -11Q242 -11 190 33L182 40V-45V-101Q182 -128 184 -134T195 -145Q216 -148 244 -148H260V-194H252L228 -193Q205 -192 178 -192T140 -191Q37 -191 28 -194H20V-148H36ZM424 218Q424 292 390 347T305 402Q234 402 182 337V98Q222 26 294 26Q345 26 384 80T424 218Z"></path><path stroke-width="1" id="MJMATHI-50" d="M287 628Q287 635 230 637Q206 637 199 638T192 648Q192 649 194 659Q200 679 203 681T397 683Q587 682 600 680Q664 669 707 631T751 530Q751 453 685 389Q616 321 507 303Q500 302 402 301H307L277 182Q247 66 247 59Q247 55 248 54T255 50T272 48T305 46H336Q342 37 342 35Q342 19 335 5Q330 0 319 0Q316 0 282 1T182 2Q120 2 87 2T51 1Q33 1 33 11Q33 13 36 25Q40 41 44 43T67 46Q94 46 127 49Q141 52 146 61Q149 65 218 339T287 628ZM645 554Q645 567 643 575T634 597T609 619T560 635Q553 636 480 637Q463 637 445 637T416 636T404 636Q391 635 386 627Q384 621 367 550T332 412T314 344Q314 342 395 342H407H430Q542 342 590 392Q617 419 631 471T645 554Z"></path><path stroke-width="1" id="MJMATHI-41" d="M208 74Q208 50 254 46Q272 46 272 35Q272 34 270 22Q267 8 264 4T251 0Q249 0 239 0T205 1T141 2Q70 2 50 0H42Q35 7 35 11Q37 38 48 46H62Q132 49 164 96Q170 102 345 401T523 704Q530 716 547 716H555H572Q578 707 578 706L606 383Q634 60 636 57Q641 46 701 46Q726 46 726 36Q726 34 723 22Q720 7 718 4T704 0Q701 0 690 0T651 1T578 2Q484 2 455 0H443Q437 6 437 9T439 27Q443 40 445 43L449 46H469Q523 49 533 63L521 213H283L249 155Q208 86 208 74ZM516 260Q516 271 504 416T490 562L463 519Q447 492 400 412L310 260L413 259Q516 259 516 260Z"></path><path stroke-width="1" id="MJMAIN-2212" d="M84 237T84 250T98 270H679Q694 262 694 250T679 230H98Q84 237 84 250Z"></path><path stroke-width="1" id="MJMATHI-45" d="M492 213Q472 213 472 226Q472 230 477 250T482 285Q482 316 461 323T364 330H312Q311 328 277 192T243 52Q243 48 254 48T334 46Q428 46 458 48T518 61Q567 77 599 117T670 248Q680 270 683 272Q690 274 698 274Q718 274 718 261Q613 7 608 2Q605 0 322 0H133Q31 0 31 11Q31 13 34 25Q38 41 42 43T65 46Q92 46 125 49Q139 52 144 61Q146 66 215 342T285 622Q285 629 281 629Q273 632 228 634H197Q191 640 191 642T193 659Q197 676 203 680H757Q764 676 764 669Q764 664 751 557T737 447Q735 440 717 440H705Q698 445 698 453L701 476Q704 500 704 528Q704 558 697 578T678 609T643 625T596 632T532 634H485Q397 633 392 631Q388 629 386 622Q385 619 355 499T324 377Q347 376 372 376H398Q464 376 489 391T534 472Q538 488 540 490T557 493Q562 493 565 493T570 492T572 491T574 487T577 483L544 351Q511 218 508 216Q505 213 492 213Z"></path><path stroke-width="1" id="MJMAIN-43" d="M56 342Q56 428 89 500T174 615T283 681T391 705Q394 705 400 705T408 704Q499 704 569 636L582 624L612 663Q639 700 643 704Q644 704 647 704T653 705H657Q660 705 666 699V419L660 413H626Q620 419 619 430Q610 512 571 572T476 651Q457 658 426 658Q322 658 252 588Q173 509 173 342Q173 221 211 151Q232 111 263 84T328 45T384 29T428 24Q517 24 571 93T626 244Q626 251 632 257H660L666 251V236Q661 133 590 56T403 -21Q262 -21 159 83T56 342Z"></path><path stroke-width="1" id="MJMAIN-6F" d="M28 214Q28 309 93 378T250 448Q340 448 405 380T471 215Q471 120 407 55T250 -10Q153 -10 91 57T28 214ZM250 30Q372 30 372 193V225V250Q372 272 371 288T364 326T348 362T317 390T268 410Q263 411 252 411Q222 411 195 399Q152 377 139 338T126 246V226Q126 130 145 91Q177 30 250 30Z"></path><path stroke-width="1" id="MJMAIN-65" d="M28 218Q28 273 48 318T98 391T163 433T229 448Q282 448 320 430T378 380T406 316T415 245Q415 238 408 231H126V216Q126 68 226 36Q246 30 270 30Q312 30 342 62Q359 79 369 104L379 128Q382 131 395 131H398Q415 131 415 121Q415 117 412 108Q393 53 349 21T250 -11Q155 -11 92 58T28 218ZM333 275Q322 403 238 411H236Q228 411 220 410T195 402T166 381T143 340T127 274V267H333V275Z"></path><path stroke-width="1" id="MJMAIN-63" d="M370 305T349 305T313 320T297 358Q297 381 312 396Q317 401 317 402T307 404Q281 408 258 408Q209 408 178 376Q131 329 131 219Q131 137 162 90Q203 29 272 29Q313 29 338 55T374 117Q376 125 379 127T395 129H409Q415 123 415 120Q415 116 411 104T395 71T366 33T318 2T249 -11Q163 -11 99 53T34 214Q34 318 99 383T250 448T370 421T404 357Q404 334 387 320Z"></path><path stroke-width="1" id="MJMAIN-74" d="M27 422Q80 426 109 478T141 600V615H181V431H316V385H181V241Q182 116 182 100T189 68Q203 29 238 29Q282 29 292 100Q293 108 293 146V181H333V146V134Q333 57 291 17Q264 -10 221 -10Q187 -10 162 2T124 33T105 68T98 100Q97 107 97 248V385H18V422H27Z"></path><path stroke-width="1" id="MJMAIN-6E" d="M41 46H55Q94 46 102 60V68Q102 77 102 91T102 122T103 161T103 203Q103 234 103 269T102 328V351Q99 370 88 376T43 385H25V408Q25 431 27 431L37 432Q47 433 65 434T102 436Q119 437 138 438T167 441T178 442H181V402Q181 364 182 364T187 369T199 384T218 402T247 421T285 437Q305 442 336 442Q450 438 463 329Q464 322 464 190V104Q464 66 466 59T477 49Q498 46 526 46H542V0H534L510 1Q487 2 460 2T422 3Q319 3 310 0H302V46H318Q379 46 379 62Q380 64 380 200Q379 335 378 343Q372 371 358 385T334 402T308 404Q263 404 229 370Q202 343 195 315T187 232V168V108Q187 78 188 68T191 55T200 49Q221 46 249 46H265V0H257L234 1Q210 2 183 2T145 3Q42 3 33 0H25V46H41Z"></path><path stroke-width="1" id="MJMAIN-48" d="M128 622Q121 629 117 631T101 634T58 637H25V683H36Q57 680 180 680Q315 680 324 683H335V637H302Q262 636 251 634T233 622L232 500V378H517V622Q510 629 506 631T490 634T447 637H414V683H425Q446 680 569 680Q704 680 713 683H724V637H691Q651 636 640 634T622 622V61Q628 51 639 49T691 46H724V0H713Q692 3 569 3Q434 3 425 0H414V46H447Q489 47 498 49T517 61V332H232V197L233 61Q239 51 250 49T302 46H335V0H324Q303 3 180 3Q45 3 36 0H25V46H58Q100 47 109 49T128 61V622Z"></path><path stroke-width="1" id="MJMAIN-69" d="M69 609Q69 637 87 653T131 669Q154 667 171 652T188 609Q188 579 171 564T129 549Q104 549 87 564T69 609ZM247 0Q232 3 143 3Q132 3 106 3T56 1L34 0H26V46H42Q70 46 91 49Q100 53 102 60T104 102V205V293Q104 345 102 359T88 378Q74 385 41 385H30V408Q30 431 32 431L42 432Q52 433 70 434T106 436Q123 437 142 438T171 441T182 442H185V62Q190 52 197 50T232 46H255V0H247Z"></path><path stroke-width="1" id="MJMAIN-54" d="M36 443Q37 448 46 558T55 671V677H666V671Q667 666 676 556T685 443V437H645V443Q645 445 642 478T631 544T610 593Q593 614 555 625Q534 630 478 630H451H443Q417 630 414 618Q413 616 413 339V63Q420 53 439 50T528 46H558V0H545L361 3Q186 1 177 0H164V46H194Q264 46 283 49T309 63V339V550Q309 620 304 625T271 630H244H224Q154 630 119 601Q101 585 93 554T81 486T76 443V437H36V443Z"></path><path stroke-width="1" id="MJMAIN-6C" d="M42 46H56Q95 46 103 60V68Q103 77 103 91T103 124T104 167T104 217T104 272T104 329Q104 366 104 407T104 482T104 542T103 586T103 603Q100 622 89 628T44 637H26V660Q26 683 28 683L38 684Q48 685 67 686T104 688Q121 689 141 690T171 693T182 694H185V379Q185 62 186 60Q190 52 198 49Q219 46 247 46H263V0H255L232 1Q209 2 183 2T145 3T107 3T57 1L34 0H26V46H42Z"></path></defs></svg></div><div id="MathJax_Message" style="display: none;"></div><div id="lightningjs-usabilla_live" style="display: none;"><div><iframe frameborder="0" id="lightningjs-frame-usabilla_live" src="./Developing a corpus of clinical notes manually annotated for part-of-speech - ScienceDirect_files/saved_resource(1).html"></iframe></div></div>
      <a class="sr-only sr-only-focusable" href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#app">Skip to main content</a>
      <!--[if lt IE 9]>
      <div id="ie8Warning" class="warning">
        <script>function ie8click() {
  const node = document.getElementById('ie8Warning');
  document.cookie = 'ie_warning_state=1';
  node.parentNode.removeChild(node);
}</script>
        <p>Please note that Internet Explorer version 8.x is not supported as of January 1, 2016.
        Please refer to <a href="https://blog.sciencedirect.com/posts/have-you-heard-spread-the-word-about-upgrading-your-browser">this blog post</a> for more information.</p>
        <a class="warning-close" onclick="ie8click()" title="Close IE warning">&times;</a>
      </div>
    <![endif]-->
      <header>
        <div class="ccs-component"><div class="header" role="banner"><div id="https-banner" class=""><div class="message">View ScienceDirect over a secure connection: <a id="https-link" class="link" href="https://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub">switch to HTTPS</a></div></div><div class="els-header"><div class="els-header-sub-nav-container " id="els-header-sub-nav-container"><div class="els-header-leftside"><div class="els-header-wordmark"><a href="http://www.sciencedirect.com/"><span aria-label="ScienceDirect" class="ccs-icon svg-wordmark-sciencedirect-orange els-main-title"><span class="svg-text">ScienceDirect</span></span></a></div></div><div class="els-header-sub-nav"><ul class="JournalsAndBooks els-header-navigation" role="navigation"><li class="els-header-navigation-section"><a id="ccs-journalsLink" class="link-inherit els-header-navigation-item " href="http://www.sciencedirect.com/science/journals">Journals</a></li><li class="els-header-navigation-section"><a id="ccs-booksLink" class="link-inherit els-header-navigation-item " href="http://www.sciencedirect.com/science/bookbshsrw">Books</a></li><li class="els-header-navigation-section"><a class="link-inherit els-header-navigation-item " href="http://www.sciencedirect.com/user/register?returnURL=%2Fscience%2Farticle%2Fpii%2FS1386505605001723">Register</a></li></ul><ul class="els-header-icons"><li class="els-header-icons-item els-header-user"><a id="ccs-signInLink" href="http://www.sciencedirect.com/user/login?returnURL=%2Fscience%2Farticle%2Fpii%2FS1386505605001723" class="link-inherit els-header-user-link"><span class="signIn_User"><span id="els-header-user-sign-in" class="els-header-user-sign-in">Sign in</span><span id="els-header-user-sign-in-icon" class="ccs-icon svg-sdfe-arrow-right els-header-user-icon-signin"></span></span></a></li><li class="Help els-header-icons-item"><a id="ccs-help-link" class="link-inherit els-header-help" href="https://service.elsevier.com/app/home/supporthub/sciencedirect/" title="Help" target="_blank"><span aria-label="Help" class="ccs-icon svg-help"><span class="svg-text">Help</span></span></a></li></ul><div class="LibraryBanner" id="library-banner"><div class="customer-banner-container"><div class="customer-banner"><span>Brought to you by:</span><a target="_blank" href="http://library.uvic.ca/" rel="nofollow"><span id="libraryBannerText">University of Victoria Libraries</span></a></div></div><div class="els-header-library-close"><a class="ccs-icon svg-delete link-inherit" id="ccs-closeLibraryBanner" aria-label="Close library banner" aria-controls="library-banner" role="button" title="Close"></a></div></div><div id="mobile-menu-parent"><div class="HamburgerButton" id="hamburger-button"><button id="ccs-mobileHamburgerButton" aria-controls="" aria-expanded="false" aria-label="Menu" class="link"><span class="svg-hamburger-closed"></span></button></div><div aria-label="Mobile menu" class="MobileMenu" id="mobile-menu"><div class="mobile-menu-container" id="mobile-menu-container"><div class="mobilemenu-content"><div><div class="primary-mobile-menu"><ul class="JournalsAndBooks els-header-navigation" role="navigation"><li class="els-header-navigation-section"><a id="mobileccs-journalsLink" class="link-inherit els-header-navigation-item " href="http://www.sciencedirect.com/science/journals">Journals</a></li><li class="els-header-navigation-section"><a id="mobileccs-booksLink" class="link-inherit els-header-navigation-item " href="http://www.sciencedirect.com/science/bookbshsrw">Books</a></li><li class="els-header-navigation-section"><a class="link-inherit els-header-navigation-item " href="http://www.sciencedirect.com/user/register?returnURL=%2Fscience%2Farticle%2Fpii%2FS1386505605001723">Register</a></li></ul></div><div class="secondary-mobile-menu"><li class="els-header-icons-item els-header-user"><a href="http://www.sciencedirect.com/user/login?returnURL=%2Fscience%2Farticle%2Fpii%2FS1386505605001723" id="mobileccs-signInLink" class="link-inherit els-header-user-link"><span class="signIn_User"><span id="els-header-user-sign-in" class="els-header-user-sign-in">Sign in</span><span id="els-header-user-sign-in-icon" class="ccs-icon svg-sdfe-arrow-right els-header-user-icon-signin"></span></span></a></li><li class="Help els-header-icons-item"><a id="mobileccs-help-link" class="link-inherit els-header-help" href="https://service.elsevier.com/app/home/supporthub/sciencedirect/" title="Help" target="_blank"><span aria-label="Help" class="ccs-icon svg-help"><span class="svg-text">Help</span></span></a></li></div></div></div></div></div></div></div></div></div></div></div>
      </header>
      <div data-iso-key="_0"><div class="App" id="app" role="application" data-reactroot=""><div class="page grid"><div class="row"><div class="col-p-3-3"><section><div class="Article"><div class="sticky-outer-wrapper active" style="height: 60px;"><div class="sticky-inner-wrapper" style="position: fixed; z-index: 1; transform: translate3d(0px, 0px, 0px); top: 0px; width: 1351px;"><div class="Toolbar" role="region" aria-label="download options and search"><div class="container"><div class="hide-m hide-t cl-l-3-12">&nbsp;</div><div class="buttons pull-left pad-left"><button class="show-toc-button hide-l hide-d"><svg focusable="false" viewBox="0 0 104 128" width="19.5" height="24" class="icon icon-list"><path d="m2e1 95a9 9 0 0 1 -9 9 9 9 0 0 1 -9 -9 9 9 0 0 1 9 -9 9 9 0 0 1 9 9zm0-3e1a9 9 0 0 1 -9 9 9 9 0 0 1 -9 -9 9 9 0 0 1 9 -9 9 9 0 0 1 9 9zm0-3e1a9 9 0 0 1 -9 9 9 9 0 0 1 -9 -9 9 9 0 0 1 9 -9 9 9 0 0 1 9 9zm14 55h68v1e1h-68zm0-3e1h68v1e1h-68zm0-3e1h68v1e1h-68z"></path></svg><span class="label">Outline</span></button><div class="PdfDownloadButton"><a id="pdfLink" class="pdf-download-btn-link" href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#" aria-controls="Download PDF options"><i class="icon-pdf"></i><span class="pdf-download-label hide-m hide-t">Download PDF</span><span class="pdf-download-label-short hide-l hide-d">Download</span></a></div><div class="ExportCitation" role="region"><button class="button ExportCitationButton button-anchor" type="button"><span class="button-text">Export</span><svg focusable="false" viewBox="0 0 92 128" width="17.25" height="24" class="icon icon-navigate-down"><path d="m1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button></div></div><div class="pull-right pad-right hide-m"><form class="QuickSearch" action="http://www.sciencedirect.com/search" method="get"><input type="text" class="query" aria-label="Search ScienceDirect" name="qs" placeholder="Search ScienceDirect"><button class="button button-primary" type="submit" aria-label="Submit search"><span class="button-text"><svg focusable="false" viewBox="0 0 100 128" height="20" width="18.75" class="icon icon-search"><path d="m19.22 76.91c-5.84-5.84-9.05-13.6-9.05-21.85s3.21-16.01 9.05-21.85c5.84-5.83 13.59-9.05 21.85-9.05 8.25 0 16.01 3.22 21.84 9.05 5.84 5.84 9.05 13.6 9.05 21.85s-3.21 16.01-9.05 21.85c-5.83 5.83-13.59 9.05-21.84 9.05-8.26 0-16.01-3.22-21.85-9.05zm80.33 29.6l-26.32-26.32c5.61-7.15 8.68-15.9 8.68-25.13 0-10.91-4.25-21.17-11.96-28.88-7.72-7.71-17.97-11.96-28.88-11.96s-21.17 4.25-28.88 11.96c-7.72 7.71-11.97 17.97-11.97 28.88s4.25 21.17 11.97 28.88c7.71 7.71 17.97 11.96 28.88 11.96 9.23 0 17.98-3.07 25.13-8.68l26.32 26.32 7.03-7.03"></path></svg></span></button><a class="advanced-search-link" href="http://www.sciencedirect.com/science/search">Advanced</a><input type="hidden" name="origin" value="article"><input type="hidden" name="zone" value="qSearch"></form></div></div></div></div></div><div class="article-wrapper"><div class="hide-m hide-t cl-l-3-12"><div class="TableOfContents" lang="en"><div class="Outline"><h2>Outline</h2><ol><li><a href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#aep-abstract-id15" title="Abstract">Abstract</a></li><li><a href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#aep-keywords-id20" title="Keywords">Keywords</a></li><li><a href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#aep-section-id21" title="1. Introduction">1. Introduction</a></li><li><a href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#aep-section-id22" title="2. Background">2. Background</a></li><li><a href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#aep-section-id23" title="3. Clinical notes and quasi-spontaneous discourse">3. Clinical notes and quasi-spontaneous discourse</a></li><li><a href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#aep-section-id24" title="4. Materials and methods">4. Materials and methods</a></li><li><a href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#aep-section-id34" title="5. Results and discussion">5. Results and discussion</a></li><li><a href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#sec1" title="6. Discussion">6. Discussion</a></li><li><a href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#aep-section-id39" title="7. Conclusion">7. Conclusion</a></li><li><a href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#aep-acknowledgment-id40" title="Acknowledgements">Acknowledgements</a></li><li><a href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#aep-bibliography-id41" title="References">References</a></li></ol><button class="button button-anchor" type="button"><span class="button-text">Show full outline</span><svg focusable="false" viewBox="0 0 92 128" height="20" width="17.25" class="icon icon-navigate-down"><path d="m1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button><div class="PageDivider"></div></div><div class="Figures"><h2>Figures (3)</h2><ol><li><a href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#fig1"><div><img alt="Fig. 1. A clinical note sample (this is a note for a non-existent test patient)" src="./Developing a corpus of clinical notes manually annotated for part-of-speech - ScienceDirect_files/1-s2.0-S1386505605001723-gr1.sml" style="max-width: 125px; max-height: 91px;"></div></a></li><li><a href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#fig2"><div><img alt="Fig. 2. XML schema that represents an annotated document" src="./Developing a corpus of clinical notes manually annotated for part-of-speech - ScienceDirect_files/1-s2.0-S1386505605001723-gr2.sml" style="max-width: 125px; max-height: 18px;"></div></a></li><li><a href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#fig3"><div><img alt="Fig. 3. A sample clinical notes text marked up in XML for POS information" src="./Developing a corpus of clinical notes manually annotated for part-of-speech - ScienceDirect_files/1-s2.0-S1386505605001723-gr3.sml" style="max-width: 125px; max-height: 82px;"></div></a></li></ol><div class="PageDivider"></div></div><div class="Tables"><h2>Tables (11)</h2><ol><li><a href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#tbl1" title="Penn Treebank tagset"><svg focusable="false" viewBox="0 0 98 128" width="18.375" height="24" class="icon icon-table"><path d="m54 68h32v32h-32v-32zm-42 0h32v32h-32v-32zm0-42h32v32h-32v-32zm42 0h32v32h-32v-32zm-52 84h94v-94h-94v94z"></path></svg>Table 1</a></li><li><a href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#tbl2" title="Annotator agreement results based on five clinical notes"><svg focusable="false" viewBox="0 0 98 128" width="18.375" height="24" class="icon icon-table"><path d="m54 68h32v32h-32v-32zm-42 0h32v32h-32v-32zm0-42h32v32h-32v-32zm42 0h32v32h-32v-32zm-52 84h94v-94h-94v94z"></path></svg>Table 2</a></li><li><a href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#tbl3" title="Absolute agreement results based on five clinical notes with an “authority” label set"><svg focusable="false" viewBox="0 0 98 128" width="18.375" height="24" class="icon icon-table"><path d="m54 68h32v32h-32v-32zm-42 0h32v32h-32v-32zm0-42h32v32h-32v-32zm42 0h32v32h-32v-32zm-52 84h94v-94h-94v94z"></path></svg>Table 3</a></li><li><a href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#tbl4" title="Syntactic category distribution in the corpus of clinical notes"><svg focusable="false" viewBox="0 0 98 128" width="18.375" height="24" class="icon icon-table"><path d="m54 68h32v32h-32v-32zm-42 0h32v32h-32v-32zm0-42h32v32h-32v-32zm42 0h32v32h-32v-32zm-52 84h94v-94h-94v94z"></path></svg>Table 4</a></li><li><a href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#tbl5" title="Correctness results for the tagger trained on Penn Treebank"><svg focusable="false" viewBox="0 0 98 128" width="18.375" height="24" class="icon icon-table"><path d="m54 68h32v32h-32v-32zm-42 0h32v32h-32v-32zm0-42h32v32h-32v-32zm42 0h32v32h-32v-32zm-52 84h94v-94h-94v94z"></path></svg>Table 5</a></li><li><a href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#tbl6" title="Correctness results for the tagger trained on the clinical notes"><svg focusable="false" viewBox="0 0 98 128" width="18.375" height="24" class="icon icon-table"><path d="m54 68h32v32h-32v-32zm-42 0h32v32h-32v-32zm0-42h32v32h-32v-32zm42 0h32v32h-32v-32zm-52 84h94v-94h-94v94z"></path></svg>Table 6</a></li></ol><button class="button button-anchor" type="button"><span class="button-text">Show all tables</span><svg focusable="false" viewBox="0 0 92 128" height="20" width="17.25" class="icon icon-navigate-down"><path d="m1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button><div class="PageDivider"></div></div>&nbsp;</div></div><article class="cl-m-3-3 cl-t-6-9 cl-l-6-12 pad-left pad-right" role="main" lang="en"><noscript>JavaScript is disabled on your browser. Please enable JavaScript to use all the features on this page.</noscript><div class="Publication"><div class="publication-brand"><a href="http://www.sciencedirect.com/science/journal/13865056"><img class="publication-brand-image hide-m" src="./Developing a corpus of clinical notes manually annotated for part-of-speech - ScienceDirect_files/elsevier-non-solus.png" alt="Elsevier"></a></div><div class="publication-volume"><h2 class="publication-title"><span class="size-xl"><a class="publication-title-link" title="Go to International Journal of Medical Informatics on ScienceDirect" href="http://www.sciencedirect.com/science/journal/13865056">International Journal of Medical Informatics</a></span></h2><div><span class="size-m"><a title="Go to table of contents for this volume/issue" href="http://www.sciencedirect.com/science/journal/13865056/75/6"><span>Volume 75, Issue 6</span></a>, <!-- -->June 2006<!-- -->, Pages 418-429</span></div></div><div class="publication-cover"><a href="http://www.sciencedirect.com/science/journal/13865056/75/6"><img class="publication-cover-image hide-m" src="./Developing a corpus of clinical notes manually annotated for part-of-speech - ScienceDirect_files/1-s2.0-S1386505606X00722-cov150h.gif" alt="International Journal of Medical Informatics"></a></div></div><h1 class="Head"><span class="title-text">Developing a corpus of clinical notes manually annotated for part-of-speech</span></h1><div class="Banner"><div class="wrapper truncated"><div class="AuthorGroups"><div class="author-group"><span class="sr-only">Author links open overlay panel</span><a class="author size-m workspace-trigger" name="baep-author-id12" href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#!"><span class="content"><span class="text given-name">Serguei V.</span><span class="text surname">Pakhomov</span><span class="author-ref" id="baff1"><sup>a</sup></span><svg focusable="false" viewBox="0 0 106 128" width="19.875" height="24" class="icon icon-person"><path d="m11.07 1.2e2l0.84-9.29c1.97-18.79 23.34-22.93 41.09-22.93 17.74 0 39.11 4.13 41.08 22.84l0.84 9.38h10.04l-0.93-10.34c-2.15-20.43-20.14-31.66-51.03-31.66s-48.89 11.22-51.05 31.73l-0.91 10.27h10.03m41.93-102.29c-9.72 0-18.24 8.69-18.24 18.59 0 13.67 7.84 23.98 18.24 23.98s18.24-10.31 18.24-23.98c0-9.9-8.52-18.59-18.24-18.59zm0 52.29c-15.96 0-28-14.48-28-33.67 0-15.36 12.82-28.33 28-28.33s28 12.97 28 28.33c0 19.19-12.04 33.67-28 33.67"></path></svg><svg focusable="false" viewBox="0 0 102 128" width="19.125" height="24" class="icon icon-envelope"><path d="m55.8 57.2c-1.78 1.31-5.14 1.31-6.9 0l-31.32-23.2h69.54l-31.32 23.19zm-55.8-24.78l42.94 32.62c2.64 1.95 6.02 2.93 9.4 2.93s6.78-0.98 9.42-2.93l40.24-30.7v-10.34h-102zm92 56.48l-18.06-22.74-8.04 5.95 17.38 21.89h-64.54l18.38-23.12-8.04-5.96-19.08 24.02v-37.58l-1e1 -8.46v61.1h102v-59.18l-1e1 8.46v35.62"></path></svg></span></a><a class="author size-m workspace-trigger" name="baep-author-id13" href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#!"><span class="content"><span class="text given-name">Anni</span><span class="text surname">Coden</span><span class="author-ref" id="baff2"><sup>b</sup></span><svg focusable="false" viewBox="0 0 102 128" width="19.125" height="24" class="icon icon-envelope"><path d="m55.8 57.2c-1.78 1.31-5.14 1.31-6.9 0l-31.32-23.2h69.54l-31.32 23.19zm-55.8-24.78l42.94 32.62c2.64 1.95 6.02 2.93 9.4 2.93s6.78-0.98 9.42-2.93l40.24-30.7v-10.34h-102zm92 56.48l-18.06-22.74-8.04 5.95 17.38 21.89h-64.54l18.38-23.12-8.04-5.96-19.08 24.02v-37.58l-1e1 -8.46v61.1h102v-59.18l-1e1 8.46v35.62"></path></svg></span></a><a class="author size-m workspace-trigger" name="baep-author-id14" href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#!"><span class="content"><span class="text given-name">Christopher G.</span><span class="text surname">Chute</span><span class="author-ref" id="baff1"><sup>a</sup></span><svg focusable="false" viewBox="0 0 102 128" width="19.125" height="24" class="icon icon-envelope"><path d="m55.8 57.2c-1.78 1.31-5.14 1.31-6.9 0l-31.32-23.2h69.54l-31.32 23.19zm-55.8-24.78l42.94 32.62c2.64 1.95 6.02 2.93 9.4 2.93s6.78-0.98 9.42-2.93l40.24-30.7v-10.34h-102zm92 56.48l-18.06-22.74-8.04 5.95 17.38 21.89h-64.54l18.38-23.12-8.04-5.96-19.08 24.02v-37.58l-1e1 -8.46v61.1h102v-59.18l-1e1 8.46v35.62"></path></svg></span></a></div></div></div><button class="ExpandingButton show-details"><svg viewBox="0 0 9 9" class="icon-expand"><path d="M5 7H4V5H2V4h2V2h1v2h2v1H5z"></path><path d="M0 0v9h9V0zm1 1h7v7H1z"></path></svg>Show more</button></div><div class="DoiLink"><a class="doi" href="https://doi.org/10.1016/j.ijmedinf.2005.08.006" target="_blank" aria-label="Persistent link using digital object identifier" title="Persistent link using digital object identifier">https://doi.org/10.1016/j.ijmedinf.2005.08.006</a><a class="rights-and-content" target="_blank" href="https://s100.copyright.com/AppDispatchServlet?publisherName=ELS&amp;contentID=S1386505605001723&amp;orderBeanReset=true">Get rights and content</a></div><section class="ReferencedArticles"></section><section class="ReferencedArticles"></section><div class="PageDivider"></div><div class="Abstracts"><div class="abstract author" id="aep-abstract-id15"><h2 class="section-title">Abstract</h2><div id="aep-abstract-sec-id16"><h3>Purpose</h3><p>This paper presents a project whose main goal is to construct a corpus of clinical text manually annotated for part-of-speech (POS) information. We describe and discuss the process of training three domain experts to perform linguistic annotation.</p></div><div id="aep-abstract-sec-id17"><h3>Methods</h3><p>Three domain experts were trained to perform manual annotation of a corpus of clinical notes. A part of this corpus was combined with the Penn Treebank corpus of general purpose English text and another part was set aside for testing. The corpora were then used for training and testing statistical part-of-speech taggers. We list some of the challenges as well as encouraging results pertaining to inter-rater agreement and consistency of annotation.</p></div><div id="aep-abstract-sec-id18"><h3>Results</h3><p>We used the Trigrams‘n’Tags (TnT) [T. Brants, TnT—a statistical part-of-speech tagger, In: Proceedings of NAACL/ANLP-2000 Symposium, 2000] tagger trained on general English data to achieve 89.79% correctness. The same tagger trained on a portion of the medical data annotated for this project improved the performance to 94.69%. Furthermore, we find that discriminating between different types of discourse represented by different sections of clinical text may be very beneficial to improve correctness of POS tagging.</p></div><div id="aep-abstract-sec-id19"><h3>Conclusion</h3><p>Our preliminary experimental results indicate the necessity for adapting state-of-the-art POS taggers to the sublanguage domain of clinical text.</p></div></div></div><ul class="issue-navigation"><li class="previous"><a class="button-alternative button-alternative-tertiary" href="http://www.sciencedirect.com/science/article/pii/S1386505605000900"><svg focusable="false" viewBox="0 0 54 128" width="32" height="32" class="icon icon-navigate-left"><path d="m1 61l45-45 7 7-38 38 38 38-7 7z"></path></svg><span class="button-alternative-text"><span><strong>Previous </strong>article<span class="extra-detail"> in issue</span></span></span></a></li><li class="next"><a class="button-alternative button-alternative-tertiary" href="http://www.sciencedirect.com/science/article/pii/S1386505605000912"><span class="button-alternative-text"><strong>Next </strong>article<span class="extra-detail"> in issue</span></span><svg focusable="false" viewBox="0 0 54 128" width="32" height="32" class="icon icon-navigate-right"><path d="m1 99l38-38-38-38 7-7 45 45-45 45z"></path></svg></a></li></ul><div class="Keywords"><div id="aep-keywords-id20" class="keywords-section"><h2 class="section-title">Keywords</h2><div class="keyword"><span>Natural language processing</span></div><div class="keyword"><span>Statistical part-of-speech tagging</span></div><div class="keyword"><span>Domain adaptation</span></div><div class="keyword"><span>Medical domain</span></div><div class="keyword"><span>Text analysis</span></div><div class="keyword"><span>Manual text annotation</span></div></div></div><div class="Body" id="body-mathjax-container"><div><section id="aep-section-id21"><h2>1. Introduction</h2><p>Natural language processing (NLP) has relatively recently emerged within Medical and Biomedical Informatics as a viable approach to processing large amounts of textual data stored in electronic form in biomedical literature as well as clinical reports. One of the goals of NLP research and development is to provide means for identifying structural elements contained within unstructured text of clinical documents. For example, the information can be structured in terms of disorders, signs or symptoms a particular patient may have as well as other facts about a patient such as whether they drink or smoke or whether a particular disorder is present in any of their family members. This is just a small sample of the kind of information that can be mined from the free text of clinical records. The obvious long-term benefit of doing so lies in the subsequent combining of this “phenotypic” data mined from text with the genomic data. NLP is critical to extracting such “phenotypic” information from text. Typically, NLP is used to provide linguistic information necessary for low level tasks such as term identification <a name="bbib1" href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#bib1" class="workspace-trigger">[1–3]</a>, word sense ambiguity resolution <a name="bbib4" href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#bib4" class="workspace-trigger">[4,17]</a>, negation recognition <a name="bbib5" href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#bib5" class="workspace-trigger">[5]</a>, as well as the higher level tasks such as automatic question answering <a name="bbib6" href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#bib6" class="workspace-trigger">[6]</a>, data mining and knowledge discovery <a name="bbib7" href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#bib7" class="workspace-trigger">[7]</a>. Many of these tasks rely on the availability of accurate POS information.</p><p>Words of any given language can be categorized according to their formal characteristics. The notion of part-of-speech (POS) pertains to such categorization. For example, certain English words can have a plural form (e.g. nouns) while others cannot (e.g. adjectives, articles, prepositions, etc.). Given a number of such observations about a language, we can classify words into large POS categories containing other words with similar characteristics. Automatic POS tagging is a task of assigning POS categories to terms from a predefined set of categories. The granularity of the POS classification system used for automatic POS tag assignment typically depends on the subsequent NLP tasks for which the POS information will subsequently be used. Some POS systems contain as few as 12 categories (NLM MetaMap tagset),<a name="bfn1" href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#fn1" class="workspace-trigger"><sup>1</sup></a> while others have 45 (Penn Treebank tagset<a name="bfn2" href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#fn2" class="workspace-trigger"><sup>2</sup></a><a name="bbib8" href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#bib8" class="workspace-trigger">[8]</a>) or over 130 (CLAWS tagset).<a name="bfn3" href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#fn3" class="workspace-trigger"><sup>3</sup></a> The higher the granularity of the tagset, the more fine grained distinctions can be made for subsequent processes; however, the tradeoff is that granular classifications introduce a greater potential for error and thus more training data are needed to maintain accuracy.</p><p>Having reliable part-of-speech (POS) information is critical to successful implementation of various natural language processing techniques for processing unrestricted text in the biomedical domain. State-of-the-art automated POS taggers achieve accuracy of 93–98% and the most successful implementations are based on statistical approaches to POS tagging. Taggers based on the Hidden Markoff Model (HMM) technology currently appear to be in the lead. Prime public domain examples of such implementations include the Trigrams‘n’Tags (TnT) tagger <a name="bbib9" href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#bib9" class="workspace-trigger">[9]</a>, Xerox tagger <a name="bbib10" href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#bib10" class="workspace-trigger">[10]</a> and LT POS tagger <a name="bbib11" href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#bib11" class="workspace-trigger">[11]</a>. Maximum Entropy (MaxEnt) based taggers also seem to perform very well <a name="bbib12" href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#bib12" class="workspace-trigger">[12,22]</a>.</p><p>One of the issues with statistical POS taggers is that most of them need a representative amount of hand-labeled training data either in the form of a comprehensive lexicon and a corpus of untagged data or a large corpus of text annotated for POS or a combination of the two. Currently, most of the POS tagger accuracy reports are based on experiments involving Penn Treebank data that consists of manually tagged corpora. These corpora include the Department of Energy abstracts, Brown corpus, Department of Agriculture bulletins, Library of America texts, MUC-3 texts, sentences from IBM computer manuals and the ATIS corpus of spontaneous speech transcriptions of Air Travel Information Systems project <a name="bbib8" href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#bib8" class="workspace-trigger">[8]</a>. The purpose of this particular selection of sources in Treebank is to represent the general English domain as completely as possible; however, it is not entirely clear how representative the general English language vocabulary and structure are of a specialized subdomain such as clinical reports. This article partly addresses this issue.</p><p>Based on the observable differences between clinical and general English discourse and POS tagging accuracy results on unknown vocabulary, it is reasonable to assume that a tagger trained on general English may not perform as well on clinical notes, where the percentage of unknown words is greater. However, in order to test this assumption, a “gold standard” corpus of clinical notes needs to be manually annotated for POS information. The issues with the annotation process constitute the primary focus of this article.</p><p>In the remainder of the article, we describe an effort to train three medical coding experts to mark the text of clinical notes for part-of-speech information. The motivation for using medical coders rather than trained linguists is three-fold. First of all, due to confidentiality restrictions, in order to develop a corpus of hand-labeled data from clinical notes one can only use personnel authorized to access patient information. The only way to avoid this restriction is to anonymize the notes prior to POS tagging which in itself is a difficult and expensive process <a name="bbib13" href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#bib13" class="workspace-trigger">[13]</a>. Second, medical coding experts are well familiar with clinical discourse, which helps especially with annotating medicine specific vocabulary. Third, the fact that POS tagging can be viewed as a classification task makes the medical coding experts highly suitable because their primary occupation and expertise is in classifying patient records for subsequent retrieval.</p><p>We discuss the training process, various issues that are due to certain specifics of the clinical notes discourse, and the results of a pilot study that evaluates the level of inter-rater agreement between the three annotators. We show that given a good set of guidelines, medical coding experts can be trained in a limited amount of time to perform a linguistic task such as POS annotation at a high level of agreement on both clinical notes and the <em>Penn Treebank Wall Street Journal</em> data subset (referred to in the rest of the paper as Penn Treebank). Finally, we report on a set of training experiments performed with the TnT tagger <a name="bbib9" href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#bib9" class="workspace-trigger">[9]</a> using Penn Treebank as well as the newly developed medical corpus.</p></section><section id="aep-section-id22"><h2>2. Background</h2><p>A well-recognized problem is that the accuracy of all current POS taggers drops dramatically on unknown words. For example, while the TnT tagger performs at 97% accuracy on known words in the Treebank, the accuracy drops to 89% on unknown words <a name="bbib9" href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#bib9" class="workspace-trigger">[9]</a>. The LT POS tagger is reported to perform at 93.6–94.3% accuracy on known words and at 87.7–88.7% on unknown words using a cascading unknown word “guesser” <a name="bbib11" href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#bib11" class="workspace-trigger">[11]</a>. The overall results for both of these taggers are much closer to the high end of the spectrum because the rate of the unknown words in the tests performed on the Penn Treebank corpus is generally relatively low—2.9% <a name="bbib9" href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#bib9" class="workspace-trigger">[9]</a>. From these results, we can derive a testable hypothesis that the higher the rate of unknown vocabulary, the lower the overall accuracy will be, necessitating adaptation of the taggers trained on Penn Treebank to sublanguage domains with vocabulary that is substantially different from that found in the Penn Treebank corpus.</p><p>A number of people have addressed this issue in the past. Campbel and Johnson <a name="bbib14" href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#bib14" class="workspace-trigger">[14]</a> show that adapting a POS tagger to medical data improves its performance. Similar results have been obtained by <a name="bbib15" href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#bib15" class="workspace-trigger">[15]</a> where a stochastic HMM-based POS tagger (Medpost) was trained on MEDLINE abstracts. Smith et al. <a name="bbib15" href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#bib15" class="workspace-trigger">[15]</a> emphasize the importance of having a wide coverage lexicon for open class words such as nouns, verbs, adjectives, etc. available to achieve high accuracy. With a medical lexicon of 10,000 words, Smith et al. <a name="bbib15" href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#bib15" class="workspace-trigger">[15]</a> are able to achieve over 97% accuracy on MEDLINE data. Another aspect of lexical items in biomedical texts – ambiguity – was investigated by Wertmer and Hahn <a name="bbib16" href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#bib16" class="workspace-trigger">[16]</a>. They find that lexical items in French biomedical corpora are less ambiguous (15.8%) overall with respect to their POS category as well as word sense than in general French text (29.1%). They also conclude that adaptation is necessary when going from a general purpose to a specialized domain.</p><p>Wertmer and Hahn <a name="bbib16" href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#bib16" class="workspace-trigger">[16]</a> find evidence to the contrary where they experiment with two statistical POS taggers (Brill and TnT) trained on German newspaper data and refute Campbel et al.'s conclusions on the necessity of generating domain specific training data and adapting POS taggers. Wertmer and Hahn <a name="bbib16" href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#bib16" class="workspace-trigger">[16]</a> applied the POS taggers to a variety of clinical documents that included discharge summaries, pathology reports and surgical reports. They find that the POS taggers trained on general German data perform very well on the medical corpus (close to 97% accuracy) and attribute this surprising result to the significant overlap in POS <em>n</em>-gram pattern types between the newspaper and the medical corpora. They conclude that statistical POS taggers trained on general purpose data can be readily used to process medical documents.</p><p>While there appears to be ample evidence to validate their conclusion for the German language, it is unclear whether it would generalize to other languages such as English. These experiments were conducted on a relatively small corpus (little over 100,000 tokens) of German text with restricted coverage—it represents only three types of clinical reports. Another important characteristic of their data that may have contributed to the surprising results is the fact that German has a very rich inflectional morphology, where the inflection in many cases tends to be a good predictor of the grammatical class to which the word belongs. The statistical package that performs the best on the medical data in their experiments is the TnT tagger <a name="bbib9" href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#bib9" class="workspace-trigger">[9]</a>, which has an algorithm for unknown (not seen in the training corpus) word category prediction. This algorithm is based on predicting the POS category based on affixation—the last <em>N</em> characters of a given unknown word. The experiments conducted by Brants <a name="bbib9" href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#bib9" class="workspace-trigger">[9]</a> on the NEGRA corpus of German newspaper articles show that the accuracy of predicting the correct category for unknown German words is 89% which is 4% higher compared to similar experiments by Brants <a name="bbib9" href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#bib9" class="workspace-trigger">[9]</a> on the Penn Treebank English data—85%.</p><p>At the very least, these numbers suggest that the TnT tagger has built-in functionality that handles unknown words in German much better than English. Brill tagger has similar functionality for generalizing to unknown words. We do not have the corresponding numbers for the Brill tagger on German data; however, the evidence obtained with the TnT tagger seems to point in the direction of German inflectional morphology possibly contributing to the surprising results found by Wertmer and Hahn <a name="bbib16" href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#bib16" class="workspace-trigger">[16]</a>. In light of these facts, it is unclear whether Wertmer and Hahn's conclusion would generalize to the English medical texts, because English has a relatively poor and underspecified morphology compared to German and thus does not lend itself as easily to the existing algorithms for unknown word category prediction.</p><p>Given the state of the art in POS tagging that we are aware of, there still appears to be a need for medical textual corpora of POS annotated data that can be used for both, testing hypotheses and tools, as well as training and adaptation of the general purpose tools to the domain. It is also important to note that the data we are working with consist of clinical notes collected at the Mayo Clinic. The language of the clinical notes is markedly different from that of pathology, surgical and radiology reports and, certainly, biomedical articles, which provides further justification for creating an annotated corpus for the clinical notes domain. We outline some of the characteristic features of clinical notes in the next section.</p></section><section id="aep-section-id23"><h2>3. Clinical notes and quasi-spontaneous discourse</h2><p>Clinical notes at the Mayo Clinic represent a record of an encounter between a physician and a patient. After each patient's visit, physicians are required by law to file a report that describes the findings and diagnoses discovered during the visit as well as other information that may be pertinent. From the language processing perspective, the main challenge presented by the clinical notes is that they represent quasi-spontaneous discourse <a name="bbib18" href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#bib18" class="workspace-trigger">[18]</a>. Clinical notes are mostly dictated from memory and sometimes from partial handwritten notes. The characteristic features of this discourse include incomplete (see example (1)) and tabular/templated statements (see examples (2) and (7)), inverted constructions (examples (3) and (4)), marginally grammatical conversational statements (see example (5)). In addition, the process of transcription creates a noisy filter where various idiosyncratic elements are introduced. These include but are not limited to misspellings, shorthand notations (see examples (8) and (9)), unsanctioned abbreviations and acronyms (see example (9)).</p><p>Some examples of statements found in clinical notes:<dl class="list"><dt class="list-label">(1)</dt><dd class="list-description"><p>Patient appreciative.</p></dd><dt class="list-label">(2)</dt><dd class="list-description"><p>ACTIVITY: as tolerated.</p></dd><dt class="list-label">(3)</dt><dd class="list-description"><p>Attending was Dr. LAST_NAME.</p></dd><dt class="list-label">(4)</dt><dd class="list-description"><p>Cataract, OS, moderate OS, stable.</p></dd><dt class="list-label">(5)</dt><dd class="list-description"><p>Today is wife's retirement party.</p></dd><dt class="list-label">(6)</dt><dd class="list-description"><p>To assist with D/C plans and eventual homegoing.</p></dd><dt class="list-label">(7)</dt><dd class="list-description"><p>Beconase nasal spray, one squirt each nostril q.d.</p></dd><dt class="list-label">(8)</dt><dd class="list-description"><p>Lacerations<a name="bfn4" href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#fn4" class="workspace-trigger"><sup>4</sup></a>&nbsp;×&nbsp;2.</p></dd><dt class="list-label">(9)</dt><dd class="list-description"><p>S/P L hip #.<a name="bfn5" href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#fn5" class="workspace-trigger"><sup>5</sup></a></p></dd></dl></p><p>In short, clinical notes combine the characteristics of conversational speech with the shorthand efficiency devices introduced by transcriptionists. This combination makes clinical note processing particularly challenging as compared to other kinds of medical texts.</p><div><p>The fact that clinical notes recorded at the Mayo Clinic are loosely structured using the HL7 Clinical Document Architecture (CDA)<a name="bfn6" href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#fn6" class="workspace-trigger"><sup>6</sup></a> specification is also important to note as it may help in some text analytic tasks including POS tagging and word sense disambiguation. <a name="bfig1" href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#fig1" class="workspace-trigger">Fig. 1</a> shows a typical clinical note. Under the CDA, clinical notes are partitioned into sections such as Chief Complaint, Current Medications, History of Present Illness, Social History, Impression/Report/Plan, among many others. One can easily imagine that, for example, the language of a Social History section would be vastly different from the language of a Current Medications section and that these two sections should probably be treated very differently from NLP standpoint.</p><figure id="fig1"><span><img src="./Developing a corpus of clinical notes manually annotated for part-of-speech - ScienceDirect_files/1-s2.0-S1386505605001723-gr1.jpg" height="520" alt="A clinical note sample (this is a note for a non-existent test patient)"><ol class="links-for-figure"><li><a href="./Developing a corpus of clinical notes manually annotated for part-of-speech - ScienceDirect_files/1-s2.0-S1386505605001723-gr1.jpg" target="__blank">Download full-size image</a></li></ol></span><div class="captions"><span><p><span class="label">Fig. 1</span>. A clinical note sample (this is a note for a non-existent test patient).</p></span></div></figure></div></section><section id="aep-section-id24"><h2>4. Materials and methods</h2><div><p>Prior to this study, the three annotators who participated in it had a substantial experience in coding clinical diagnoses but virtually no experience in POS markup. The training process consisted of a general and rather superficial introduction to the issues in linguistics as well as some formal training using the POS tagging guidelines developed by Santoriny <a name="bbib19" href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#bib19" class="workspace-trigger">[19]</a> for tagging Penn Treebank data. <a name="btbl1" href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#tbl1" class="workspace-trigger">Table 1</a> displays all the POS tags present in the Penn Treebank data and their short descriptions.</p><div class="tables frame-topbot rowsep-0 colsep-0" id="tbl1"><div class="captions"><span><p><span class="label">Table 1</span>. Penn Treebank tagset</p></span></div><div class="groups"><table><thead class="valign-top"><tr><th scope="col" class="rowsep-1 align-left">Tag</th><th scope="col" class="rowsep-1 align-left">Description</th><th scope="col" class="rowsep-1 align-left">Tag</th><th scope="col" class="rowsep-1 align-left">Description</th></tr></thead><tbody><tr><td class="align-left">CC</td><td class="align-left">Coordinating conjunction</td><td class="align-left">POS</td><td class="align-left">Possessive ending</td></tr><tr><td class="align-left">CD</td><td class="align-left">Cardinal number</td><td class="align-left">PRP</td><td class="align-left">Personal pronoun</td></tr><tr><td class="align-left">DT</td><td class="align-left">Determiner</td><td class="align-left">PRP$</td><td class="align-left">Possessive pronoun</td></tr><tr><td class="align-left">EX</td><td class="align-left">Existential there</td><td class="align-left">RB</td><td class="align-left">Adverb</td></tr><tr><td class="align-left">FW</td><td class="align-left">Foreign word</td><td class="align-left">RBR</td><td class="align-left">Adverb, comparative</td></tr><tr><td class="align-left">IN</td><td class="align-left">Preposition/subordinating conjunction</td><td class="align-left">RBS</td><td class="align-left">Adverb, superlative</td></tr><tr><td class="align-left">JJ</td><td class="align-left">Adjective</td><td class="align-left">RP</td><td class="align-left">Particle</td></tr><tr><td class="align-left">JJR</td><td class="align-left">Adjective, comparative</td><td class="align-left">TO</td><td class="align-left">To</td></tr><tr><td class="align-left">JJS</td><td class="align-left">Adjective, superlative</td><td class="align-left">UH</td><td class="align-left">Interjection</td></tr><tr><td class="align-left">LS</td><td class="align-left">List marker</td><td class="align-left">VB</td><td class="align-left">Verb, base form</td></tr><tr><td class="align-left">MD</td><td class="align-left">Modal</td><td class="align-left">VBD</td><td class="align-left">Verb, past tense</td></tr><tr><td class="align-left">NN</td><td class="align-left">Noun, singular or mass</td><td class="align-left">VBG</td><td class="align-left">Verb, gerund/present participle</td></tr><tr><td class="align-left">NNS</td><td class="align-left">Noun plural</td><td class="align-left">VBN</td><td class="align-left">Verb, past participle</td></tr><tr><td class="align-left">NNP</td><td class="align-left">Proper noun, singular</td><td class="align-left">VBP</td><td class="align-left">Verb, singular, present, non-3D</td></tr><tr><td class="align-left">NNPS</td><td class="align-left">Proper noun, plural</td><td class="align-left">VBZ</td><td class="align-left">Verb, 3rd person singular, present</td></tr><tr><td class="align-left">PDT</td><td class="align-left">Predeterminer</td><td class="align-left">WDT</td><td class="align-left">Wh-determiner</td></tr><tr><td></td><td></td><td class="align-left">WP</td><td class="align-left">Wh-pronoun</td></tr></tbody></table></div></div></div><p>The formal training was followed by informal discussions of the data and difficult cases pertinent to the clinical notes domain. The discussions often resulted in slight modifications to the Penn Treebank guidelines. Below is a list of such cases and the modifications.</p><section id="aep-section-id25"><h3>4.1. Special symbols</h3><p>Medical transcriptionists often use a single keystroke shorthand for words that occur relatively frequently. For example, “+” is often used to mean “positive”, as in “positive throat culture”, “−” for “negative throat culture.” The pound sign “#” is often used to mean “pounds” or “fracture”, “x” is often used to mean “scar” as in “x 2”—two scars. It is not entirely clear at this point if it would be more beneficial to treat these symbols as actual symbols (SYM) or as a special kind of abbreviations. In the latter case, they would be tagged as if the actual word was used in place of the symbol. For the time being we rather arbitrarily decided to mark them as symbols SYM—“x/SYM 2/CD.”</p></section><section id="aep-section-id26"><h3>4.2. Drug names</h3><p>Drug names are tagged as singular proper nouns (NNP) regardless of whether they are capitalized. Numeric and other types of attributes of drug names are considered to be part of the drug name and are tagged as NNP as well (Ex. “Extra/NNP Strength/NNP Tylenol/NNP #4/NNP”). The choice of the NNP label was fairly arbitrary and largely driven by informal observation that drug names appear to be similar to other named entities such as geographical locations and persons in their distribution and orthography. The choice was also partly driven by the desire to avoid introducing new tags for the sake of future compatibility with other NLP tools. At a later point the token “#4” in this example and other examples of a symbol followed by a numeral, were converted to two tokens “#” and the numeral and re-tagged as a symbol (SYM) and cardinal number (CD), respectively. It can be argued that this decision may adversely affect the results of an experiment where a tagger trained on the Penn Treebank data is tested on the clinical notes data; however, our experiments show that such effect is unlikely. We address this issue in more detail in Section <a name="bsec1" href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#sec1" class="workspace-trigger">6</a>.</p></section><section id="aep-section-id27"><h3>4.3. Dosages</h3><p>Dosages have a recognizable structure, which in most cases, consists of the numeric magnitude of the dose (e.g. 500), the measurement unit (e.g. mg), method of delivery (e.g. orally) and the interval of delivery (e.g. b.i.d., twice a day, etc.). The amount is tagged as a cardinal number (CD), the measurement unit is tagged as NNS or NN depending on whether the measurements are of a plural or singular entity, the method of delivery is tagged the same as normal text. A more problematic case is the interval of delivery, which may include actual phrases “twice daily” or their Latin equivalents “b.i.d.” The former is tagged as if it were normal text, the latter is tagged as a foreign word—FW.</p></section><section id="aep-section-id28"><h3>4.4. Foreign words</h3><p>The medical domain is permeated with words of Latin and Greek origin and making a clear distinction between a medical word of foreign origin and a medical word that is a foreign word is rather difficult even in cases where the word has retained its foreign pronunciation, e.g. “polymyalgia rheumatica.” In the majority of such cases, the words in question are somehow related to a condition, procedure or some other clinically related phenomenon and normally not used separately. Based on that, we tag potential candidates for foreign words as if they formed a unit. By this token, “polymyalgia rheumatica” is tagged as a noun followed by another noun and forms a compound noun—polymyalgia/NN rheumatica/NN.</p></section><section id="aep-section-id29"><h3>4.5. Annotation structure and format</h3><div><p>Each clinical note is represented as an Extensible Markup Language (XML) document with the underlying schema shown in <a name="bfig2" href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#fig2" class="workspace-trigger">Fig. 2</a>. The raw clinical notes go through automatic POS tagging using the Maximum Entropy POS tagger <a name="bbib22" href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#bib22" class="workspace-trigger">[22]</a> whose results are rendered in XML format and are then presented in a graphical XML editor for correction.</p><figure id="fig2"><span><img src="./Developing a corpus of clinical notes manually annotated for part-of-speech - ScienceDirect_files/1-s2.0-S1386505605001723-gr2.jpg" height="87" alt="XML schema that represents an annotated document"><ol class="links-for-figure"><li><a href="./Developing a corpus of clinical notes manually annotated for part-of-speech - ScienceDirect_files/1-s2.0-S1386505605001723-gr2.jpg" target="__blank">Download full-size image</a></li></ol></span><div class="captions"><span><p><span class="label">Fig. 2</span>. XML schema that represents an annotated document.</p></span></div></figure></div><div><p>The top node in the XML schema represents the whole document and can have a number of “section” nodes under it that represent various subsections of a clinical note such as History of Present Illness (HPI), Chief Complaint (CC) and other standard sections compliant with HL-7 Clinical Document Architecture specification. The section nodes branch out into sentence nodes and each sentence node can branch out directly into <em>W</em> nodes, which are combinations of word orthographic representation and its part-of-speech information. The <em>W</em> nodes can be arranged into phrases with various types of attributes. These can be either traditional linguistic phrases such as NP, VP, etc. or specialized medical phrases such as “drug mention.” The “phrase” nodes group <em>W</em> nodes under it into meaningful chunks. The “meaningfulness” of the chunks as well as their boundaries is left up to the domain experts to determine. The “phrase” node also allows simultaneous anonymization of data by introducing “phrase” nodes with special attributes to group patient identifying information. <a name="bfig3" href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#fig3" class="workspace-trigger">Fig. 3</a> shows what the actual text of the note looks like after it has been marked up with XML according to the schema given in <a name="bfig2" href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#fig2" class="workspace-trigger">Fig. 2</a>.</p><figure id="fig3"><span><img src="./Developing a corpus of clinical notes manually annotated for part-of-speech - ScienceDirect_files/1-s2.0-S1386505605001723-gr3.gif" height="312" alt="A sample clinical notes text marked up in XML for POS information"><ol class="links-for-figure"><li><a href="./Developing a corpus of clinical notes manually annotated for part-of-speech - ScienceDirect_files/1-s2.0-S1386505605001723-gr3.gif" target="__blank">Download full-size image</a></li></ol></span><div class="captions"><span><p><span class="label">Fig. 3</span>. A sample clinical notes text marked up in XML for POS information.</p></span></div></figure></div><p>The “dual residence” of the <em>W</em> node under both the “phrase” node and the “sentence” node in the schema reflects the fact that we are aiming at working with incomplete shallow parses rather than full parses. In the former case, not all phrases that comprise a sentence have to be identified and, therefore, not every word has to be part of a phrase.</p><p>The data presented to the annotators is preprocessed before annotation. The pre-processing includes sentence boundary detection, tokenization and priming with part-of-speech tags generated by a MaxEnt tagger (Maxent 1.2.4 package (Baldridge et al.)) trained on Penn Treebank data. To expedite the annotation process, an in-house Java-based editor with a graphical user interface was developed.</p></section><section id="aep-section-id30"><h3>4.6. Annotator agreement</h3><p>In order to establish the reliability of the data, we need to ensure internal as well as external consistency of the annotation. First of all, we need to make sure that the annotators agree amongst themselves (internal consistency) on how they mark up text for part-of-speech information. Second, we need to find out how closely the annotators generating data for this study agree with the annotators of an established project such as Penn Treebank (external consistency). If both tests show relatively high levels of agreement, then we can safely assume that the annotators in this study are able to generate part-of-speech tags for biomedical data that will be consistent with a widely recognized standard and can work independently of each other thus tripling the amount of manually annotated data.</p></section><section id="aep-section-id31"><h3>4.7. Methods</h3><p>Two types of measures of consistency were computed—absolute agreement and Kappa coefficient. The absolute agreement was calculated according to the following formula in <a name="beq1" href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#eq1" class="workspace-trigger">(1)</a>:<span class="display"><div id="eq1" class="formula"><span class="label">(1)</span><span class="math"><span class="MathJax_Preview" style="display: none;"></span><span class="MathJax_SVG" id="MathJax-Element-1-Frame" tabindex="0" style="font-size: 90%; display: inline-block; position: relative;" data-mathml="&lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot;&gt;&lt;mrow is=&quot;true&quot;&gt;&lt;mtext is=&quot;true&quot;&gt;Abs&lt;/mtext&gt;&lt;mi is=&quot;true&quot;&gt;&lt;/mi&gt;&lt;mtext is=&quot;true&quot;&gt;agr&lt;/mtext&gt;&lt;mo is=&quot;true&quot;&gt;=&lt;/mo&gt;&lt;mn is=&quot;true&quot;&gt;100&lt;/mn&gt;&lt;mo is=&quot;true&quot;&gt;&amp;#xD7;&lt;/mo&gt;&lt;mfenced open=&quot;(&quot; close=&quot;)&quot; is=&quot;true&quot;&gt;&lt;mrow is=&quot;true&quot;&gt;&lt;mfrac is=&quot;true&quot;&gt;&lt;mi is=&quot;true&quot;&gt;M&lt;/mi&gt;&lt;mi is=&quot;true&quot;&gt;T&lt;/mi&gt;&lt;/mfrac&gt;&lt;/mrow&gt;&lt;/mfenced&gt;&lt;/mrow&gt;&lt;/math&gt;" role="presentation"><svg xmlns:xlink="http://www.w3.org/1999/xlink" width="21.307ex" height="3.323ex" viewBox="0 -970.1 9174 1430.7" role="img" focusable="false" style="vertical-align: -1.07ex;" aria-hidden="true"><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="matrix(1 0 0 -1 0 0)"><g is="true"><g is="true"><use href="#MJMAIN-41"></use><use href="#MJMAIN-62" x="750" y="0"></use><use href="#MJMAIN-73" x="1307" y="0"></use></g><g is="true"></g><g is="true" transform="translate(1701,0)"><use href="#MJMAIN-61"></use><use href="#MJMAIN-67" x="500" y="0"></use><use href="#MJMAIN-72" x="1001" y="0"></use></g><g is="true" transform="translate(3372,0)"><use href="#MJMAIN-3D"></use></g><g is="true" transform="translate(4429,0)"><use href="#MJMAIN-31"></use><use href="#MJMAIN-30" x="500" y="0"></use><use href="#MJMAIN-30" x="1001" y="0"></use></g><g is="true" transform="translate(6152,0)"><use href="#MJMAIN-D7"></use></g><g is="true" transform="translate(7153,0)"><use href="#MJSZ1-28"></use><g is="true" transform="translate(458,0)"><g is="true"><g transform="translate(120,0)"><rect stroke="none" width="863" height="60" x="0" y="220"></rect><g is="true" transform="translate(60,406)"><use transform="scale(0.707)" href="#MJMATHI-4D"></use></g><g is="true" transform="translate(182,-385)"><use transform="scale(0.707)" href="#MJMATHI-54"></use></g></g></g></g><use href="#MJSZ1-29" x="1562" y="-1"></use></g></g></g></svg><span class="MJX_Assistive_MathML" role="presentation"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow is="true"><mtext is="true">Abs</mtext><mi is="true"></mi><mtext is="true">agr</mtext><mo is="true">=</mo><mn is="true">100</mn><mo is="true">×</mo><mfenced open="(" close=")" is="true"><mrow is="true"><mfrac is="true"><mi is="true">M</mi><mi is="true">T</mi></mfrac></mrow></mfenced></mrow></math></span></span><script type="math/mml" id="MathJax-Element-1"><math><mrow is="true"><mtext is="true">Abs</mtext><mi is="true"></mi><mtext is="true">agr</mtext><mo is="true">=</mo><mn is="true">100</mn><mo is="true">×</mo><mfenced open="(" close=")" is="true"><mrow is="true"><mfrac is="true"><mi is="true">M</mi><mi is="true">T</mi></mfrac></mrow></mfenced></mrow></math></script></span></div></span>where <em>M</em> is the total number of time all annotators agreed on a tag and <em>T</em> is the total number of tags.</p><p>Kappa coefficient is given in <a name="beq2" href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#eq2" class="workspace-trigger">(2)</a><a name="bbib20" href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#bib20" class="workspace-trigger">[20]</a>:<span class="display"><div id="eq2" class="formula"><span class="label">(2)</span><span class="math"><span class="MathJax_Preview" style="display: none;"></span><span class="MathJax_SVG" id="MathJax-Element-2-Frame" tabindex="0" style="font-size: 90%; display: inline-block; position: relative;" data-mathml="&lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot;&gt;&lt;mrow is=&quot;true&quot;&gt;&lt;mtext is=&quot;true&quot;&gt;Kappa&lt;/mtext&gt;&lt;mo is=&quot;true&quot;&gt;=&lt;/mo&gt;&lt;mfrac is=&quot;true&quot;&gt;&lt;mrow is=&quot;true&quot;&gt;&lt;mi is=&quot;true&quot;&gt;P&lt;/mi&gt;&lt;mo stretchy=&quot;false&quot; is=&quot;true&quot;&gt;(&lt;/mo&gt;&lt;mi is=&quot;true&quot;&gt;A&lt;/mi&gt;&lt;mo stretchy=&quot;false&quot; is=&quot;true&quot;&gt;)&lt;/mo&gt;&lt;mo is=&quot;true&quot;&gt;&amp;#x2212;&lt;/mo&gt;&lt;mi is=&quot;true&quot;&gt;P&lt;/mi&gt;&lt;mo stretchy=&quot;false&quot; is=&quot;true&quot;&gt;(&lt;/mo&gt;&lt;mi is=&quot;true&quot;&gt;E&lt;/mi&gt;&lt;mo stretchy=&quot;false&quot; is=&quot;true&quot;&gt;)&lt;/mo&gt;&lt;/mrow&gt;&lt;mrow is=&quot;true&quot;&gt;&lt;mn is=&quot;true&quot;&gt;1&lt;/mn&gt;&lt;mo is=&quot;true&quot;&gt;&amp;#x2212;&lt;/mo&gt;&lt;mi is=&quot;true&quot;&gt;P&lt;/mi&gt;&lt;mo stretchy=&quot;false&quot; is=&quot;true&quot;&gt;(&lt;/mo&gt;&lt;mi is=&quot;true&quot;&gt;E&lt;/mi&gt;&lt;mo stretchy=&quot;false&quot; is=&quot;true&quot;&gt;)&lt;/mo&gt;&lt;/mrow&gt;&lt;/mfrac&gt;&lt;/mrow&gt;&lt;/math&gt;" role="presentation"><svg xmlns:xlink="http://www.w3.org/1999/xlink" width="19.446ex" height="4.388ex" viewBox="0 -1173.9 8372.8 1889.3" role="img" focusable="false" style="vertical-align: -1.661ex;" aria-hidden="true"><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="matrix(1 0 0 -1 0 0)"><g is="true"><g is="true"><use href="#MJMAIN-4B"></use><use href="#MJMAIN-61" x="778" y="0"></use><use href="#MJMAIN-70" x="1279" y="0"></use><use href="#MJMAIN-70" x="1835" y="0"></use><use href="#MJMAIN-61" x="2392" y="0"></use></g><g is="true" transform="translate(3170,0)"><use href="#MJMAIN-3D"></use></g><g is="true" transform="translate(3948,0)"><g transform="translate(397,0)"><rect stroke="none" width="3906" height="60" x="0" y="220"></rect><g is="true" transform="translate(60,583)"><g is="true"><use transform="scale(0.707)" href="#MJMATHI-50"></use></g><g is="true" transform="translate(531,0)"><use transform="scale(0.707)" href="#MJMAIN-28"></use></g><g is="true" transform="translate(806,0)"><use transform="scale(0.707)" href="#MJMATHI-41"></use></g><g is="true" transform="translate(1337,0)"><use transform="scale(0.707)" href="#MJMAIN-29"></use></g><g is="true" transform="translate(1612,0)"><use transform="scale(0.707)" href="#MJMAIN-2212"></use></g><g is="true" transform="translate(2163,0)"><use transform="scale(0.707)" href="#MJMATHI-50"></use></g><g is="true" transform="translate(2694,0)"><use transform="scale(0.707)" href="#MJMAIN-28"></use></g><g is="true" transform="translate(2970,0)"><use transform="scale(0.707)" href="#MJMATHI-45"></use></g><g is="true" transform="translate(3510,0)"><use transform="scale(0.707)" href="#MJMAIN-29"></use></g></g><g is="true" transform="translate(689,-437)"><g is="true"><use transform="scale(0.707)" href="#MJMAIN-31"></use></g><g is="true" transform="translate(353,0)"><use transform="scale(0.707)" href="#MJMAIN-2212"></use></g><g is="true" transform="translate(904,0)"><use transform="scale(0.707)" href="#MJMATHI-50"></use></g><g is="true" transform="translate(1435,0)"><use transform="scale(0.707)" href="#MJMAIN-28"></use></g><g is="true" transform="translate(1711,0)"><use transform="scale(0.707)" href="#MJMATHI-45"></use></g><g is="true" transform="translate(2251,0)"><use transform="scale(0.707)" href="#MJMAIN-29"></use></g></g></g></g></g></g></svg><span class="MJX_Assistive_MathML" role="presentation"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow is="true"><mtext is="true">Kappa</mtext><mo is="true">=</mo><mfrac is="true"><mrow is="true"><mi is="true">P</mi><mo stretchy="false" is="true">(</mo><mi is="true">A</mi><mo stretchy="false" is="true">)</mo><mo is="true">−</mo><mi is="true">P</mi><mo stretchy="false" is="true">(</mo><mi is="true">E</mi><mo stretchy="false" is="true">)</mo></mrow><mrow is="true"><mn is="true">1</mn><mo is="true">−</mo><mi is="true">P</mi><mo stretchy="false" is="true">(</mo><mi is="true">E</mi><mo stretchy="false" is="true">)</mo></mrow></mfrac></mrow></math></span></span><script type="math/mml" id="MathJax-Element-2"><math><mrow is="true"><mtext is="true">Kappa</mtext><mo is="true">=</mo><mfrac is="true"><mrow is="true"><mi is="true">P</mi><mo stretchy="false" is="true">(</mo><mi is="true">A</mi><mo stretchy="false" is="true">)</mo><mo is="true">−</mo><mi is="true">P</mi><mo stretchy="false" is="true">(</mo><mi is="true">E</mi><mo stretchy="false" is="true">)</mo></mrow><mrow is="true"><mn is="true">1</mn><mo is="true">−</mo><mi is="true">P</mi><mo stretchy="false" is="true">(</mo><mi is="true">E</mi><mo stretchy="false" is="true">)</mo></mrow></mfrac></mrow></math></script></span></div></span>where <em>P</em>(<em>A</em>) is the proportion of times the annotators actually agree and <em>P</em>(<em>E</em>) is the proportion of times the annotators are expected to agree due to chance.<a name="bfn7" href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#fn7" class="workspace-trigger"><sup>7</sup></a></p><p>The absolute agreement is most informative when computed over several sets of labels and where one of the sets represents the “authoritative” set. In this case, the ratio of matches among all the sets including the “authoritative” set to the total number of labels shows how close the other sets are to the “authoritative” one. The Kappa-statistic is useful in measuring how consistent the annotators are compared to each other as opposed to an authority standard.</p></section><section id="aep-section-id32"><h3>4.8. Internal consistency</h3><p>In order to test for internal consistency, we analyzed inter-annotator agreement where the three annotators tagged the same small corpus of clinical dictations.</p><div><p>The results were compared and the Kappa-statistic was used to calculate the inter-annotator agreement. The results of this experiment are summarized in <a name="btbl2" href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#tbl2" class="workspace-trigger">Table 2</a>. For the absolute agreement, we computed the ratio of how many times all three annotators agreed on a tag for a given token to the total number of tags.</p><div class="tables frame-topbot rowsep-0 colsep-0" id="tbl2"><div class="captions"><span><p><span class="label">Table 2</span>. Annotator agreement results based on five clinical notes</p></span></div><div class="groups"><table><thead class="valign-top"><tr><th scope="col" class="rowsep-1 align-left">Note ID</th><th scope="col" class="rowsep-1 align-left">Abs agr (%)</th><th scope="col" class="rowsep-1 align-left">Kappa</th><th scope="col" class="rowsep-1 align-left"><em>N</em> samples</th></tr></thead><tbody><tr><td class="align-left">1137689</td><td class="align-left">93.24</td><td class="align-left">0.9527</td><td class="align-char">755</td></tr><tr><td class="align-left">1165875</td><td class="align-left">94.59</td><td class="align-left">0.9622</td><td class="align-char">795</td></tr><tr><td class="align-left">1283904</td><td class="align-left">89.79</td><td class="align-left">0.9302</td><td class="align-char">392</td></tr><tr><td class="align-left">1284881</td><td class="align-left">90.42</td><td class="align-left">0.9328</td><td class="align-char">397</td></tr><tr><td class="align-left">1307526</td><td class="align-left">84.43</td><td class="align-left">0.8943</td><td class="align-char">347</td></tr><tr><td class="align-left" colspan="4"><br></td></tr><tr><td class="align-left">Total</td><td></td><td></td><td class="align-char">2686</td></tr><tr><td class="align-left">Average</td><td class="align-left">90.49</td><td class="align-left">0.9344</td><td></td></tr></tbody></table></div></div></div><p>Based on the small pilot sample of five clinical notes (2686 words), the Kappa test showed a very high agreement coefficient—0.93. An acceptable agreement for most NLP classification tasks lies between 0.7 and 0.8 <a name="bbib20" href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#bib20" class="workspace-trigger">[20,21]</a>. Absolute agreement numbers are consistent with high Kappa as they show an average of 90% of all tags in the test documents assigned exactly the same way by all three annotators.</p></section><section id="aep-section-id33"><h3>4.9. External consistency</h3><p>The external consistency with the Penn Treebank annotation was computed using a small sample of 939 words from the Penn Treebank-2 WSJ Corpus annotated for POS information.</p><p>The sample had not been seen by the annotators prior to the test. The annotators were asked to make POS judgments on the WSJ corpus sample just as they would on the clinical notes.</p><div><p>The labels were compared to the Penn Treebank annotation individually by annotator and the results of the comparison are presented in <a name="btbl3" href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#tbl3" class="workspace-trigger">Table 3</a>. The results indicate that the three annotators who participated in this project are on average 88% consistent with the annotators of the Penn Treebank corpus.</p><div class="tables frame-topbot rowsep-0 colsep-0" id="tbl3"><div class="captions"><span><p><span class="label">Table 3</span>. Absolute agreement results based on five clinical notes with an “authority” label set</p></span></div><div class="groups"><table><thead class="valign-top"><tr><th scope="col" class="rowsep-1 align-left">Annotator</th><th scope="col" class="rowsep-1 align-left">Abs agr (%)</th></tr></thead><tbody><tr><td class="align-left">A1</td><td class="align-left">88.17</td></tr><tr><td class="align-left">A2</td><td class="align-left">87.85</td></tr><tr><td class="align-left">A3</td><td class="align-left">87.85</td></tr><tr><td class="align-left" colspan="2"><br></td></tr><tr><td class="align-left">Average</td><td class="align-left">87.95</td></tr></tbody></table></div></div></div></section></section><section id="aep-section-id34"><h2>5. Results and discussion</h2><div><p>The annotation process resulted in a corpus of 273 clinical notes annotated with POS tags. The corpus contains 100,650 tokens from 8702 types distributed across 7299 sentences. <a name="btbl4" href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#tbl4" class="workspace-trigger">Table 4</a> displays frequency counts for the top most frequent syntactic categories.</p><div class="tables frame-topbot rowsep-0 colsep-0" id="tbl4"><div class="captions"><span><p><span class="label">Table 4</span>. Syntactic category distribution in the corpus of clinical notes</p></span></div><div class="groups"><table><thead class="valign-top"><tr><th scope="col" class="rowsep-1 align-left">Category</th><th scope="col" class="rowsep-1 align-left">Count</th><th scope="col" class="rowsep-1 align-left">Total (%)</th></tr></thead><tbody><tr><td class="align-left">NN</td><td class="align-char">18372</td><td class="align-char">18</td></tr><tr><td class="align-left">IN</td><td class="align-char">8963</td><td class="align-char">9</td></tr><tr><td class="align-left">JJ</td><td class="align-char">8851</td><td class="align-char">9</td></tr><tr><td class="align-left">DT</td><td class="align-char">6796</td><td class="align-char">7</td></tr><tr><td class="align-left">NNP</td><td class="align-char">4794</td><td class="align-char">5</td></tr></tbody></table></div></div></div><p>The distribution of syntactic categories suggests the predominance of nominal categories, which is consistent with the nature of clinical notes reporting on various patient characteristics such as disorders, signs and symptoms.</p><p>Another important descriptive characteristic of this corpus is that the average sentence length is 13.79 tokens per sentence, which is relatively short as compared to the Penn Treebank corpus where the sentence length is 24.16 tokens per sentence. This supports our informal observation of the clinical notes data containing multiple sentence fragments and short diagnostic statements. Shorter sentence length implies greater number of inter-sentential transitions and therefore is likely to present a challenge for a tagger based on <em>n</em>-gram transitions such as an HMM tagger.</p><section id="aep-section-id35"><h3>5.1. Results of training and testing a POS tagger on medical data</h3><p>In order to test some of our assumptions regarding how the differences between the general English language and the language of clinical notes may affect POS tagging, we have trained the HMM-based TnT tagger <a name="bbib9" href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#bib9" class="workspace-trigger">[9]</a> with default parameters at the tri-gram level both on Penn Treebank and the clinical notes data. We should also note that the tagger relies on a sophisticated “unknown” word-guessing algorithm, which computes the likelihood of a tag based on the <em>N</em> last letters of the word, which is meant to leverage the word's morphology in a purely statistical manner.</p><p>The clinical notes data were split at random 10 times in 80/20 fashion where 80% of the sentences were used for training and 20% were used for testing. This technique is a variation on the classic 10-fold cross-validation and appears to be more suitable for smaller amounts of data. As a result, we produced 10 training clinical notes corpora and 10 testing clinical notes corpora.</p><p>We conducted two experiments. First, we trained a tagger on the Treebank data and tested it on each of the 10 testing clinical notes corpora. We tested the tagger trained on Treebank on the 10 testing corpora rather than the whole corpus of clinical notes in order to produce correctness results on exactly the same test data as would be used for testing the tagger trained on the 10 clinical notes training corpora.</p><div><p>Correctness was computed simply as the percentage of correct tag assignments of the POS tagger (hits) to the total number of tokens in the test set:<span class="display"><div class="formula"><span class="label">(3)</span><span class="math"><span class="MathJax_Preview" style="display: none;"></span><span class="MathJax_SVG" id="MathJax-Element-3-Frame" tabindex="0" style="font-size: 90%; display: inline-block; position: relative;" data-mathml="&lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot;&gt;&lt;mrow is=&quot;true&quot;&gt;&lt;mtext is=&quot;true&quot;&gt;Correctness&lt;/mtext&gt;&lt;mo is=&quot;true&quot;&gt;=&lt;/mo&gt;&lt;mn is=&quot;true&quot;&gt;100&lt;/mn&gt;&lt;mo is=&quot;true&quot;&gt;&amp;#xD7;&lt;/mo&gt;&lt;mfrac is=&quot;true&quot;&gt;&lt;mrow is=&quot;true&quot;&gt;&lt;mtext is=&quot;true&quot;&gt;Hits&lt;/mtext&gt;&lt;/mrow&gt;&lt;mrow is=&quot;true&quot;&gt;&lt;mtext is=&quot;true&quot;&gt;Total&lt;/mtext&gt;&lt;/mrow&gt;&lt;/mfrac&gt;&lt;/mrow&gt;&lt;/math&gt;" role="presentation"><svg xmlns:xlink="http://www.w3.org/1999/xlink" width="25.981ex" height="3.323ex" viewBox="0 -970.1 11186 1430.7" role="img" focusable="false" style="vertical-align: -1.07ex;" aria-hidden="true"><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="matrix(1 0 0 -1 0 0)"><g is="true"><g is="true"><use href="#MJMAIN-43"></use><use href="#MJMAIN-6F" x="722" y="0"></use><use href="#MJMAIN-72" x="1223" y="0"></use><use href="#MJMAIN-72" x="1615" y="0"></use><use href="#MJMAIN-65" x="2008" y="0"></use><use href="#MJMAIN-63" x="2452" y="0"></use><use href="#MJMAIN-74" x="2897" y="0"></use><use href="#MJMAIN-6E" x="3286" y="0"></use><use href="#MJMAIN-65" x="3843" y="0"></use><use href="#MJMAIN-73" x="4287" y="0"></use><use href="#MJMAIN-73" x="4682" y="0"></use></g><g is="true" transform="translate(5354,0)"><use href="#MJMAIN-3D"></use></g><g is="true" transform="translate(6410,0)"><use href="#MJMAIN-31"></use><use href="#MJMAIN-30" x="500" y="0"></use><use href="#MJMAIN-30" x="1001" y="0"></use></g><g is="true" transform="translate(8134,0)"><use href="#MJMAIN-D7"></use></g><g is="true" transform="translate(8912,0)"><g transform="translate(342,0)"><rect stroke="none" width="1811" height="60" x="0" y="220"></rect><g is="true" transform="translate(264,414)"><g is="true"><use transform="scale(0.707)" href="#MJMAIN-48"></use><use transform="scale(0.707)" href="#MJMAIN-69" x="750" y="0"></use><use transform="scale(0.707)" href="#MJMAIN-74" x="1029" y="0"></use><use transform="scale(0.707)" href="#MJMAIN-73" x="1418" y="0"></use></g></g><g is="true" transform="translate(60,-398)"><g is="true"><use transform="scale(0.707)" href="#MJMAIN-54"></use><use transform="scale(0.707)" href="#MJMAIN-6F" x="722" y="0"></use><use transform="scale(0.707)" href="#MJMAIN-74" x="1223" y="0"></use><use transform="scale(0.707)" href="#MJMAIN-61" x="1612" y="0"></use><use transform="scale(0.707)" href="#MJMAIN-6C" x="2113" y="0"></use></g></g></g></g></g></g></svg><span class="MJX_Assistive_MathML" role="presentation"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow is="true"><mtext is="true">Correctness</mtext><mo is="true">=</mo><mn is="true">100</mn><mo is="true">×</mo><mfrac is="true"><mrow is="true"><mtext is="true">Hits</mtext></mrow><mrow is="true"><mtext is="true">Total</mtext></mrow></mfrac></mrow></math></span></span><script type="math/mml" id="MathJax-Element-3"><math><mrow is="true"><mtext is="true">Correctness</mtext><mo is="true">=</mo><mn is="true">100</mn><mo is="true">×</mo><mfrac is="true"><mrow is="true"><mtext is="true">Hits</mtext></mrow><mrow is="true"><mtext is="true">Total</mtext></mrow></mfrac></mrow></math></script></span></div></span><a name="btbl5" href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#tbl5" class="workspace-trigger">Table 5</a> summarizes the results of testing the tagger trained on Penn Treebank data, while <a name="btbl6" href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#tbl6" class="workspace-trigger">Table 6</a> summarizes the testing results for the tagger trained on the clinical notes.</p><div class="tables frame-topbot rowsep-0 colsep-0" id="tbl5"><div class="captions"><span><p><span class="label">Table 5</span>. Correctness results for the tagger trained on Penn Treebank</p></span></div><div class="groups"><table><thead class="valign-top"><tr><th scope="col" class="rowsep-1 align-left">Split</th><th scope="col" class="rowsep-1 align-left">Hits</th><th scope="col" class="rowsep-1 align-left">Total</th><th scope="col" class="rowsep-1 align-left">Correctness (%)</th></tr></thead><tbody><tr><td class="align-char">1</td><td class="align-char">21560</td><td class="align-left">23872</td><td class="align-left">90.32</td></tr><tr><td class="align-char">2</td><td class="align-char">22122</td><td class="align-left">24665</td><td class="align-left">89.69</td></tr><tr><td class="align-char">3</td><td class="align-char">21417</td><td class="align-left">23923</td><td class="align-left">89.52</td></tr><tr><td class="align-char">4</td><td class="align-char">21970</td><td class="align-left">24461</td><td class="align-left">89.82</td></tr><tr><td class="align-char">5</td><td class="align-char">22079</td><td class="align-left">24665</td><td class="align-left">89.52</td></tr><tr><td class="align-char">6</td><td class="align-char">21649</td><td class="align-left">24049</td><td class="align-left">90.02</td></tr><tr><td class="align-char">7</td><td class="align-char">21598</td><td class="align-left">24040</td><td class="align-left">89.84</td></tr><tr><td class="align-char">8</td><td class="align-char">21379</td><td class="align-left">23882</td><td class="align-left">89.52</td></tr><tr><td class="align-char">9</td><td class="align-char">22131</td><td class="align-left">24610</td><td class="align-left">89.93</td></tr><tr><td class="align-char">10</td><td class="align-char">22358</td><td class="align-left">24923</td><td class="align-left">89.71</td></tr><tr><td class="align-left" colspan="4"><br></td></tr><tr><td class="align-left">Average</td><td class="align-char">21826.3</td><td class="align-left">24309</td><td class="align-left">89.79</td></tr></tbody></table></div></div><div class="tables frame-topbot rowsep-0 colsep-0" id="tbl6"><div class="captions"><span><p><span class="label">Table 6</span>. Correctness results for the tagger trained on the clinical notes</p></span></div><div class="groups"><table><thead class="valign-top"><tr><th scope="col" class="rowsep-1 align-left">Split</th><th scope="col" class="rowsep-1 align-left">Hits</th><th scope="col" class="rowsep-1 align-left">Total</th><th scope="col" class="rowsep-1 align-left">Correctness (%)</th></tr></thead><tbody><tr><td class="align-char">1</td><td class="align-char">22654</td><td class="align-left">23872</td><td class="align-left">94.90</td></tr><tr><td class="align-char">2</td><td class="align-char">23332</td><td class="align-left">24665</td><td class="align-left">94.60</td></tr><tr><td class="align-char">3</td><td class="align-char">22645</td><td class="align-left">23923</td><td class="align-left">94.66</td></tr><tr><td class="align-char">4</td><td class="align-char">23206</td><td class="align-left">24461</td><td class="align-left">94.87</td></tr><tr><td class="align-char">5</td><td class="align-char">23326</td><td class="align-left">24665</td><td class="align-left">94.57</td></tr><tr><td class="align-char">6</td><td class="align-char">22732</td><td class="align-left">24049</td><td class="align-left">94.52</td></tr><tr><td class="align-char">7</td><td class="align-char">22807</td><td class="align-left">24040</td><td class="align-left">94.87</td></tr><tr><td class="align-char">8</td><td class="align-char">22603</td><td class="align-left">23882</td><td class="align-left">94.64</td></tr><tr><td class="align-char">9</td><td class="align-char">23316</td><td class="align-left">24610</td><td class="align-left">94.74</td></tr><tr><td class="align-char">10</td><td class="align-char">23563</td><td class="align-left">24923</td><td class="align-left">94.54</td></tr><tr><td class="align-left" colspan="4"><br></td></tr><tr><td class="align-left">Average</td><td class="align-char">23018.4</td><td class="align-left">24309</td><td class="align-left">94.69</td></tr></tbody></table></div></div></div><p>The average correctness of the tagger trained on Penn Treebank and tested on clinical notes is ∼90%, which is considerably lower than the state-of-the-art performance of the TnT tagger, ∼96%. Training the tagger on a relatively small amount of clinical notes data brings the performance much closer to the state-of-the-art, ∼95%.</p><div><p>Given the diversity of language used in various sections of the clinical notes, we also wanted to investigate whether we would find fluctuations in accuracy results depending on the section. In order to do this, we split the corpus of medical data into 10 subcorpora, each containing data from a single section. The choice of the 10 corpora was dictated by the amount of available data. The token counts for each subcorpus are given in <a name="btbl7" href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#tbl7" class="workspace-trigger">Table 7</a>. The smallest corpus (ALLergies) contains 392 tokens, while the largest corpus (IP/impression-plan) contains 43,633 tokens. We applied a model trained on Penn Treebank to each of the 10 corpora in turn. This model is distributed with the TnT tagger package. The results are presented in <a name="btbl7" href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#tbl7" class="workspace-trigger">Table 7</a> and sorted in the increasing order of correctness, where Current Medications (CM), Special Instructions (SI) and Allergies (ALL) sections have the lowest correctness and History of Present Illness (HPI), Social History (SH) and Family History (FH) have the highest correctness. The results for these sections are greater than the mean by more than one standard deviation. Clearly, since the IP and HPI sections have the most representation in the corpus, the overall results of testing a Penn Treebank tagger on the entire medical corpus reported in <a name="btbl5" href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#tbl5" class="workspace-trigger">Table 5</a> (90%) tend to be aligned somewhere between the results obtained from the IP section (87.4%) and the HPI section (90.34%). We would like to emphasize the point that the results obtained on each individual section vary to a large extent, which suggests that further adaptation with more training data for a particular section of a clinical note may be desirable or even necessary as the correctness numbers with the CM, SI and ALL sections seem to indicate.</p><div class="tables frame-topbot rowsep-0 colsep-0" id="tbl7"><div class="captions"><span><p><span class="label">Table 7</span>. Correctness results for the general English model tested on separate sections of the medical corpus</p></span></div><div class="groups"><table><thead class="valign-top"><tr><th scope="col" class="rowsep-1"></th><th scope="col" class="rowsep-1 align-left">Token ambiguity</th><th scope="col" class="rowsep-1 align-left">Known token ambiguity</th><th scope="col" class="rowsep-1 align-left">Percent unknown</th><th scope="col" class="rowsep-1 align-left">Correctness (%)</th><th scope="col" class="rowsep-1 align-left">Token count</th></tr></thead><tbody><tr><td class="align-left">CM (Current Medications)</td><td class="align-left">5.87</td><td class="align-left">1.9</td><td class="align-char">33.22</td><td class="align-left">74.70</td><td class="align-char">3853</td></tr><tr><td class="align-left">SI (Special Instructions)</td><td class="align-left">7.03</td><td class="align-left">2.41</td><td class="align-char">23.26</td><td class="align-left">77.20</td><td class="align-char">1741</td></tr><tr><td class="align-left">ALL (Allergies)</td><td class="align-left">4.51</td><td class="align-left">1.99</td><td class="align-char">27.30</td><td class="align-left">79.34</td><td class="align-char">392</td></tr><tr><td class="align-left">PSH (Past Surgical History)</td><td class="align-left">3.77</td><td class="align-left">2.07</td><td class="align-char">22.12</td><td class="align-left">84.42</td><td class="align-char">3061</td></tr><tr><td class="align-left">CC (Chief Complaint)</td><td class="align-left">3.67</td><td class="align-left">2.12</td><td class="align-char">19.39</td><td class="align-left">86.32</td><td class="align-char">2997</td></tr><tr><td class="align-left">IP (Impression/Plan)</td><td class="align-left">3.51</td><td class="align-left">2.33</td><td class="align-char">14.67</td><td class="align-left">87.40</td><td class="align-char">43633</td></tr><tr><td class="align-left">ROS (Review of Systems)</td><td class="align-left">3.04</td><td class="align-left">2.12</td><td class="align-char">12.31</td><td class="align-left">89.25</td><td class="align-char">3469</td></tr><tr><td class="align-left">HPI (History of Present Illness)</td><td class="align-left">3.31</td><td class="align-left">2.3</td><td class="align-char">11.38</td><td class="align-left">90.34</td><td class="align-char">37262</td></tr><tr><td class="align-left">SH (Social History)</td><td class="align-left">2.61</td><td class="align-left">2.18</td><td class="align-char">5.64</td><td class="align-left">91.36</td><td class="align-char">1400</td></tr><tr><td class="align-left">FH (Family History)</td><td class="align-left">2.92</td><td class="align-left">2.1</td><td class="align-char">9.25</td><td class="align-left">92.62</td><td class="align-char">854</td></tr><tr><td class="align-left" colspan="6"><br></td></tr><tr><td class="align-left">Total</td><td></td><td></td><td></td><td></td><td class="align-char">98662</td></tr><tr><td class="align-left">Average</td><td class="align-left">4.024</td><td class="align-left">2.152</td><td class="align-char">17.85</td><td class="align-left">85.30</td><td></td></tr></tbody></table></div></div></div><p>The results presented in <a name="btbl7" href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#tbl7" class="workspace-trigger">Table 7</a> also illustrate the correlation between the percentage of tokens previously unseen in the training data (‘percent unknown’ column) and the correctness of the POS tagger. The two variables negatively correlate at −0.94. The correlation between correctness and the overall POS ambiguity (‘token ambiguity’ column) in the corpus is also very high −0.91; however, the correlation between correctness and the ambiguity of only the known (previously seen in the training data) tokens is 0.27. This observation is interesting because it indicates that there is a relationship between unknown tokens and ambiguity in terms of how both of these factors affect POS performance. In fact, one could view a token that the POS tagger never ‘saw’ in the training data as one that is as ambiguous as the entire POS tagset. Since the tagger has no statistical information about it, there is nothing to restrict the set of possible POS tags. In the case of the Penn Treebank tagset, an unknown token could be potentially ambiguous in 45 ways where the only restriction is the prior probability of each POS tag.</p><div><p>Since the top five most frequent POS categories in the corpus of clinical notes are nouns (NN), prepositions (IN), adjectives (JJ), determiners (DT) and proper nouns (NNP) as shown in <a name="btbl4" href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#tbl4" class="workspace-trigger">Table 4</a>, one would expect that the majority of the errors involving unknown words would result in one of these top categories being assigned to the unknown token. We tested this hypothesis by determining the relative frequency of the top 10 most frequent POS tags assigned to the unknown tokens. The results are summarized in <a name="btbl8" href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#tbl8" class="workspace-trigger">Table 8</a> and confirm the hypothesis. The highest relative frequency of POS tags assigned to unknown tokens belongs to nouns and proper nouns (both at 31%), followed by adjectives (17%). After adjectives, the relative frequency drops to 7% with plural nouns (NNS). The relative frequency of prepositions (IN) and determiners (DT) did not even make it into the top 10 list and is less than 1%. This is not unexpected because both DT and IN belong to a closed class of POS tags and are relatively rarely assigned erroneously.</p><div class="tables frame-topbot rowsep-0 colsep-0" id="tbl8"><div class="captions"><span><p><span class="label">Table 8</span>. Relative frequency of POS categories assigned to “unknown” tokens by the TnT tagger</p></span></div><div class="groups"><table><thead class="valign-top"><tr><th scope="col" class="rowsep-1 align-left"><em>N</em></th><th scope="col" class="rowsep-1 align-left">POS tag</th><th scope="col" class="rowsep-1 align-left">Relative frequency (%)</th></tr></thead><tbody><tr><td class="align-char">154</td><td class="align-left">VB</td><td class="align-char">1.03</td></tr><tr><td class="align-char">163</td><td class="align-left">RB</td><td class="align-char">1.09</td></tr><tr><td class="align-char">174</td><td class="align-left">VBN</td><td class="align-char">1.16</td></tr><tr><td class="align-char">175</td><td class="align-left">VBG</td><td class="align-char">1.17</td></tr><tr><td class="align-char">228</td><td class="align-left">FW</td><td class="align-char">1.52</td></tr><tr><td class="align-char">626</td><td class="align-left">CD</td><td class="align-char">4.17</td></tr><tr><td class="align-char">1003</td><td class="align-left">NNS</td><td class="align-char">6.68</td></tr><tr><td class="align-char">2693</td><td class="align-left">JJ</td><td class="align-char">17.93</td></tr><tr><td class="align-char">4736</td><td class="align-left">NNP</td><td class="align-char">31.53</td></tr><tr><td class="align-char">4738</td><td class="align-left">NN</td><td class="align-char">31.54</td></tr></tbody></table></div></div></div></section></section><section id="sec1"><h2>6. Discussion</h2><p>This paper was intended to share the goals and challenges experienced during annotation and analysis of a small sample of clinical notes data for part-of-speech information. The annotation was performed by experts in the domain of indexing medical content who are minimally trained to label medical texts for part-of-speech. We have outlined some of the challenging issues in clinical note annotation as well as some of the more outstanding differences between clinical notes and other types of written and spoken discourse widely used in training NLP applications.</p><p>The results of this pilot project are encouraging. It is clear that with appropriate supervision, people who are well familiar with medical content can be reliably trained to carry out some of the tasks traditionally done by trained linguists. We have shown that the three annotators in this study have been able to achieve relatively high levels of inter-rater agreement (Kappa ∼0.93) as well as compliance with an authoritative Penn Treebank annotation (absolute agreement ∼89%).</p><p>This study also indicates that a statistical POS tagger trained on data that does not include clinical documents may not perform as well as a tagger trained on at least some data from the clinical domain. There are a number of factors that contribute to this. The “unknown” out-of-vocabulary items is only one of the factors responsible for the decreased accuracy of POS taggers trained on the general English domain when they are used in the clinical notes domain. The differences in the actual distribution of tokens and POS categories in general English and the clinical domain seem to be another major contributing factor. A comparison between the Treebank and the clinical notes data shows that the clinical notes corpus contains 3239 lexical items that are not found in Penn Treebank. The Penn Treebank corpus contains over 40,000 lexical items that are not found in the corpus of clinical notes. 5463 lexical items are found in both corpora. In addition to this 37% out-of-vocabulary rate (words in clinical notes but not the Penn Treebank corpus), the picture is further complicated by the differences between the <em>n</em>-gram tag transitions within the two corpora. For example, the likelihood of a DT&nbsp;→&nbsp;NN bigram is 1 in Penn Treebank and 0.75 in the clinical notes corpus. On the other hand, JJ&nbsp;→&nbsp;NN transition in the clinical notes is 1 but in the Penn Treebank corpus it has a likelihood of 0.73.</p><p>We also addressed the issue of having lowercased medication names annotated as proper nouns by comparing errors produced by the two taggers. The confusion between NN and NNP produced by the tagger trained on Penn Treebank and tested on the clinical notes accounted for 14% of all errors as opposed to 6% of the errors when the tagger is tested on Penn Treebank data. This result suggests that our decision to use NNP tag for all medication names including the lowercased ones may have introduced some confusion to the “unknown” word guessing built into the TnT tagger. In order to test this hypothesis, we changed the tags of all lowercase words that were initially tagged as NNP to NN and tested the tagger trained on Penn Treebank data again on the clinical notes. This resulted in NN–NNP confusion being responsible for 12% of the errors. The overall correctness results remained unaffected—89.99%, which indicates that the NN–NNP confusion is likely due to the fact that most medication names are not in the Penn Treebank's vocabulary. Since most of them do not comply with regular English morphology an unknown word guessing algorithm is likely to do poorly on this particular category of words.</p><p>Finally, interesting results emerged from our experiments applying the general purpose POS tagger to various sections of the clinical notes. We are seeing lower than the overall correctness on SI, CM and ALL sections and higher than the overall correctness on HPI, FH and SH sections. When we examine the contents of the different sections, we find that SI, CM and ALL differ substantially from HPI, FH and SH sections in both vocabulary and structure. For example, the SI section contains a lot of names of local places around the Mayo Clinic and tends to be telegraphic in format:<span class="display"><div class="tables frame-none rowsep-0 colsep-0" id="aep-table-id36"><div class="groups"><table><tbody><tr><td class="align-left">SI:</td><td class="align-left">TESTING ONLY 10-8-98</td></tr><tr><td></td><td class="align-left">To correspondence pending biopsy</td></tr><tr><td></td><td class="align-left">Please have iron studies routed to secretary</td></tr><tr><td></td><td class="align-left">(e-mail sent)</td></tr></tbody></table></div></div></span></p><p>A typical CM section contains medication names, dosages and instructions mostly in an abbreviated form:<span class="display"><div class="tables frame-none rowsep-0 colsep-0" id="aep-table-id37"><div class="groups"><table><tbody><tr><td class="align-left">CM:</td><td class="align-left">Glucotrol XL 10&nbsp;mg one by mouth two times a day</td></tr><tr><td></td><td class="align-left">Folic acid 1&nbsp;mg one p.o. q.d.</td></tr></tbody></table></div></div></span></p><p>In contrast, while HPI, FH and SH sections also contain specialized medical vocabulary, they tend to resemble regular English one may find in print or in casual conversation much closer than CM, ALL and SI sections. Here is an example of the SH section:<span class="display"><div class="tables frame-none rowsep-0 colsep-0" id="aep-table-id38"><div class="groups"><table><tbody><tr><td class="align-left">SH:</td><td class="align-left">She drinks approximately one to two caffeinated beverages a day</td></tr><tr><td></td><td class="align-left">She does not use tobacco, alcohol, recreational, or street drugs</td></tr><tr><td></td><td class="align-left">She denies any history of abuse</td></tr><tr><td></td><td class="align-left">She wears seatbelts 100% of the time</td></tr></tbody></table></div></div></span></p><p>These examples are fairly representative of the content of the various sections in clinical notes. One possible explanation for why the results are so much better on SH, FH and HPI sections compared to CM, SI and ALL sections is that the vocabulary and the language patterns in the former are much closer to the vocabulary and the language patters found in general English data. Due to a relatively small size of the available hand-labeled data, it is hard to draw final conclusions. However, it is clear at this point that, at least for the domain of clinical notes, it is necessary to obtain domain specific data in order to train state-of-the-art POS taggers.</p></section><section id="aep-section-id39"><h2>7. Conclusion</h2><p>We have trained three medical index coders to annotate a relatively small corpus of clinical notes at the Mayo Clinic for POS information. We have also performed a series of experiments with a state-of-the-art part-of-speech tagger (TnT) trained on both general English and on a portion of the annotated clinical notes corpus. We find that using domain data are beneficial to improving POS tagging correctness and, furthermore, adaptation to various discourse types represented in the sections of clinical documents may be of substantial benefit also. Several questions remain unresolved. First of all, it is unclear how much domain specific data are enough to achieve state-of-the-art performance on POS tagging. Second, given that it is somewhat easier to develop lexicons for POS tagging than to annotate corpora, we need to find out how important the corpus statistics are as opposed to a domain specific lexicon. In other words, the question is whether we can achieve state-of-the-art performance in a specialized domain by simply adding the vocabulary from the domain to the POS tagger's lexicon. We intend to address both of these questions with further experimentation.</p></section></div><section id="aep-acknowledgment-id40"><h2 class="section-title">Acknowledgements</h2><p>We would like to thank our medical index experts Barbara Abbot, Pauline Funk and Debora Albrecht for their persistent efforts in the difficult task of corpus annotation. This work was done in part under the NLM Training grant (# T15 LM07041-19<strong>)</strong>.</p></section></div><div class="related-content-links show-t-s hide-t hide-l hide-d"><button class="button button-anchor" type="button"><span class="button-text">Special issue articles</span></button><button class="button button-anchor" type="button"><span class="button-text">Recommended articles</span></button><button class="button button-anchor" type="button"><span class="button-text">Citing articles (<!-- -->23<!-- -->)</span></button></div><div class="References"><section class="bibliography" id="aep-bibliography-id41"><h2 class="section-title">References</h2><section class="bibliography-sec" id="aep-bibliography-sec-id42"><dl class="references"><dt class="label"><a href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#bbib1" id="ref-id-bib1">[1]</a></dt><dd class="reference"><div class="contribution">G. Savova, M.R. Harris, T. Johnson, S.V. Pakhomov, C.G. Chute<strong class="title">A data-driven approach for extracting “the most specific term” for ontology development</strong></div><div class="host">Proceedings of the AMIA Symposium (2003), pp. 579-583</div><div class="ReferenceLinks"></div></dd><dt class="label"><a href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#bbib2" id="ref-id-bib2">[2]</a></dt><dd class="reference"><div class="contribution">S. Ananiadou<strong class="title">A methodology for automatic term recognition</strong></div><div class="host">Proceedings of COLING Symposium (1994), pp. 1034-1038</div><div class="ReferenceLinks"></div></dd><dt class="label"><a href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#bbib3" id="ref-id-bib3">[3]</a></dt><dd class="reference"><div class="contribution">K. Frantzi, S. Ananiadou, J. Tsujii<strong class="title">The <em>C</em>-value/NC-value method of automatic recognition for multi-word terms</strong></div><div class="host">Proceedings of ECDL Symposium (1998), pp. 585-604</div><div class="ReferenceLinks"></div></dd><dt class="label"><a href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#bbib4" id="ref-id-bib4">[4]</a></dt><dd class="reference"><div class="contribution">M. Weeber, J. Mork, A. Aronson<strong class="title">Developing a test collection for biomedical word sense disambiguation</strong></div><div class="host">Proceedings of AMIA Symposium (2001)</div><div class="ReferenceLinks"></div></dd><dt class="label"><a href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#bbib5" id="ref-id-bib5">[5]</a></dt><dd class="reference"><div class="contribution">W.W. Chapman, W. Bridewell, P. Hanbury, G.F. Cooper, B. Buchanan<strong class="title">Evaluation of negation phrases in narrative clinical reports</strong></div><div class="host">Proceedings of AMIA Symposium (2001), pp. 105-109</div><div class="ReferenceLinks"></div></dd><dt class="label"><a href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#bbib6" id="ref-id-bib6">[6]</a></dt><dd class="reference"><div class="contribution">D. Moldovan, S. Harabagiu, M. Pasca, R. Mihalcea, R. Girju, R. Goodrum, V. Rus<strong class="title">The structure and performance of an open-domain question answering system</strong></div><div class="host">Proceedings of ACL Symposium (2000)</div><div class="ReferenceLinks"></div></dd><dt class="label"><a href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#bbib7" id="ref-id-bib7">[7]</a></dt><dd class="reference"><div class="contribution">M. Hearst<strong class="title">Untangling text data mining</strong></div><div class="host">Proceedings of the 37th Annual Meeting of the Association for Computer Linguistics (ACL’99) (1999), pp. 3-10</div><div class="ReferenceLinks"></div></dd><dt class="label"><a href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#bbib8" id="ref-id-bib8">[8]</a></dt><dd class="reference"><div class="contribution">M. Marcus, B. Santorini, M.A. Marcinkiewicz<strong class="title">Building a large annotated corpus of English: the Penn Treebank</strong></div><div class="host">Comput. Linguistics, 19 (1993), pp. 297-352</div><div class="ReferenceLinks"></div></dd><dt class="label"><a href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#bbib9" id="ref-id-bib9">[9]</a></dt><dd class="reference"><div class="contribution">T. Brants<strong class="title">TnT—a statistical part-of-speech tagger</strong></div><div class="host">Proceedings of NAACL/ANLP-2000 Symposium (2000)</div><div class="ReferenceLinks"></div></dd><dt class="label"><a href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#bbib10" id="ref-id-bib10">[10]</a></dt><dd class="reference"><div class="contribution">D. Cutting, J. Kupiec, J. Pedersen, P.A. Sibun<strong class="title">Practical POS tagger</strong></div><div class="host">Proceedings of ANLP Symposium (1992)</div><div class="ReferenceLinks"></div></dd><dt class="label"><a href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#bbib11" id="ref-id-bib11">[11]</a></dt><dd class="reference"><div class="contribution">A. Mikheev<strong class="title">Automatic rule induction for unknown-word guessing</strong></div><div class="host">Comput. Linguistics, 23 (3) (1997), pp. 405-423</div><div class="ReferenceLinks"><a class="link" target="_blank" rel="noopener noreferrer" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-0005496287&amp;partnerID=10&amp;rel=R3.0.0">View Record in Scopus</a></div></dd><dt class="label"><a href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#bbib12" id="ref-id-bib12">[12]</a></dt><dd class="reference"><div class="contribution">A. Ratnaparkhi<strong class="title">A maximum entropy part of speech tagger</strong></div><div class="host">Proceedings of the Conference on Empirical Methods in Natural Language Processing (EMNLP), University of Pennsylvania, May (1996)</div><div class="ReferenceLinks"></div></dd><dt class="label"><a href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#bbib13" id="ref-id-bib13">[13]</a></dt><dd class="reference"><div class="contribution">P. Ruch, R. Baud, A. Rassinoux, P. Bouillon, G. Robert<strong class="title">Medical document anonymization with a semantic lexicon</strong></div><div class="host">Proceedings of AMIA Symposium (2000), pp. 729-733</div><div class="ReferenceLinks"></div></dd><dt class="label"><a href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#bbib14" id="ref-id-bib14">[14]</a></dt><dd class="reference"><div class="contribution">D. Campbell, S.B. Johnson<strong class="title">Comparing syntactic complexity in medical and non-medical corpora</strong></div><div class="host">Proceedings of the AMIA Annual Symposium (2001), pp. 90-95</div><div class="ReferenceLinks"></div></dd><dt class="label"><a href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#bbib15" id="ref-id-bib15">[15]</a></dt><dd class="reference"><div class="contribution">L.H. Smith, T. Rindflesch, J. Wilbur<strong class="title">MedPost: a part of speech tagger for biomedical text</strong></div><div class="host">Bioinformatics, 20 (14) (2004), pp. 2320-2321</div><div class="ReferenceLinks"><a class="link" target="_blank" rel="noopener noreferrer" href="https://doi.org/10.1093/bioinformatics/bth227">CrossRef</a><a class="link" target="_blank" rel="noopener noreferrer" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-5044241106&amp;partnerID=10&amp;rel=R3.0.0">View Record in Scopus</a></div></dd><dt class="label"><a href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#bbib16" id="ref-id-bib16">[16]</a></dt><dd class="reference"><div class="contribution">J. Wertmer, U. Hahn<strong class="title">Really, is medical sublanguage that different? Experimental counter-evidence from tagging medical and newspaper corpora</strong></div><div class="host">Proceedings of Medinfo (2004)</div><div class="ReferenceLinks"></div></dd><dt class="label"><a href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#bbib17" id="ref-id-bib17">[17]</a></dt><dd class="reference"><div class="contribution">P. Ruch, R. Baud, A. Geissbuhler, A.M. Rassinoux<strong class="title">Comparing general and medical texts for information retrieval based on natural language processing: an inquiry into lexical disambiguation</strong></div><div class="host">Proceedings of Medinfo Symposium (2001), pp. 261-265</div><div class="ReferenceLinks"></div></dd><dt class="label"><a href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#bbib18" id="ref-id-bib18">[18]</a></dt><dd class="reference"><div class="contribution">S. Pakhomov, M. Schonwetter, J. Bachenko<strong class="title">Generating training data for medical dictations</strong></div><div class="host">Proceedings of NAACL Symposium (2001)</div><div class="ReferenceLinks"></div></dd><dt class="label"><a href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#bbib19" id="ref-id-bib19">[19]</a></dt><dd class="reference"><span>B. Santorini, Part-of-speech tagging guidelines for the Penn Treebank project, Technical Report, Department of Computer and Information Science, University of Pennsylvania, 1991.</span></dd><dt class="label"><a href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#bbib20" id="ref-id-bib20">[20]</a></dt><dd class="reference"><div class="contribution">J. Carletta<strong class="title">Assessing agreement on classification tasks: the Kappa statistic</strong></div><div class="host">Comput. Linguistics, 22 (2) (1996), pp. 249-254</div><div class="ReferenceLinks"><a class="link" target="_blank" rel="noopener noreferrer" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-77949291063&amp;partnerID=10&amp;rel=R3.0.0">View Record in Scopus</a></div></dd><dt class="label"><a href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#bbib21" id="ref-id-bib21">[21]</a></dt><dd class="reference"><div class="contribution">M. Poessio, R. Vieira<strong class="title">A corpus based investigation of definite description use</strong></div><div class="host">Comput. Linguistics, 24 (2) (1988), pp. 186-215</div><div class="ReferenceLinks"></div></dd><dt class="label"><a href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#bbib22" id="ref-id-bib22">[22]</a></dt><dd class="reference"><span>J. Baldridge, T. Morton, G. Bierner, URL: <a href="http://maxent.sourceforge.net/" target="_blank">http://maxent.sourceforge.net</a> (last accessed 11/24/04).</span></dd></dl></section></section></div><div class="Footnotes"><dl class="footnote"><dt class="footnote-label"><sup><a href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#bfn1">1</a></sup></dt><dd><p><a href="http://mmtx.nlm.nih.gov/" target="_blank">http://mmtx.nlm.nih.gov/</a>.</p></dd></dl><dl class="footnote"><dt class="footnote-label"><sup><a href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#bfn2">2</a></sup></dt><dd><p>Penn Treebank tagset can also be said to have 36 tags if one does not count punctuation.</p></dd></dl><dl class="footnote"><dt class="footnote-label"><sup><a href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#bfn3">3</a></sup></dt><dd><p><a href="http://www.comp.lancs.ac.uk/ucrel/claws/" target="_blank">http://www.comp.lancs.ac.uk/ucrel/claws/</a>.</p></dd></dl><dl class="footnote"><dt class="footnote-label"><sup><a href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#bfn4">4</a></sup></dt><dd><p>“Two lacerations”.</p></dd></dl><dl class="footnote"><dt class="footnote-label"><sup><a href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#bfn5">5</a></sup></dt><dd><p>“Status post left hip fracture”.</p></dd></dl><dl class="footnote"><dt class="footnote-label"><sup><a href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#bfn6">6</a></sup></dt><dd><p>Health Level 7 is a medical standards organization part of whose purpose is to establish and maintain various standards applicable to medical information management.</p></dd></dl><dl class="footnote"><dt class="footnote-label"><sup><a href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#bfn7">7</a></sup></dt><dd><p>A very detailed explanation of the terms used in the formula for Kappa computation as well as concrete examples of how it is computed are provided in Poessio and Vieira <a name="bbib21" href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#bib21" class="workspace-trigger">[21]</a>.</p></dd></dl></div><div class="Copyright"><span class="copyright-line">Copyright © 2005 Elsevier Ireland Ltd. All rights reserved.</span></div></article><div class="hide-m hide-t-s cl-t-3-9 cl-l-3-12 pad-right"><aside class="RelatedContent" aria-label="Related content"><section class="SpecialIssueArticles"><div class="part-of-issue"><h2 class="part-of-issue-text">Part of special issue:</h2><div><a class="part-of-issue-title" href="http://www.sciencedirect.com/science/journal/13865056/75/6"><span>Recent Advances in Natural Language Processing for Biomedical Applications Special Issue</span></a><div class="part-of-issue-editors"><span>Edited by </span><div class="authors"><span>Robert</span> <span>Baud</span>, <span>Nigel</span> <span>Collier</span>, <span>Adeline</span> <span>Nazarenko</span>, <span>Patrick</span> <span>Ruch</span></div></div></div><button class="button-alternative DownloadFullIssue button-alternative-primary" type="button"><svg focusable="false" viewBox="0 0 54 128" width="32" height="32" class="icon icon-navigate-right"><path d="m1 99l38-38-38-38 7-7 45 45-45 45z"></path></svg><span class="button-alternative-text">Download full issue</span></button></div><section class="SidePanel"><header class="side-panel-header" id="special-issue-articles-header"><a class="arrow-btn head rotated" href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#" aria-controls="special-issue-articles" aria-expanded="true"><span class="Icon ArrowDown" style="display: inline-block; height: 16px; width: 16px; margin-right: 8px; transition: transform 0.3s, -webkit-transform 0.3s; transform: scaleY(-1);"><svg fill="currentColor" tabindex="-1" focusable="false" height="16" width="16" viewBox="0 0 16 16" style="width: 100%; height: 100%;"><path d="M16 4.452l-1.26-1.26L8 9.932l-6.74-6.74L0 4.452l8 8 8-8z"></path></svg></span><h2><span class="section-title">Other articles from this issue</span></h2></a></header><div id="special-issue-articles" aria-hidden="false"><ul><li class="SidePanelItem"><div class="sub-heading"><a href="http://www.sciencedirect.com/science/article/pii/S1386505605000912" data-hack="#"><h3 class="article-title ellipsis" id="special-issue-articles-article0-title"><span>Evaluation of two dependency parsers on biomedical corpus targeted at protein–protein interactions</span></h3></a><div class="article-source ellipsis">International Journal of Medical Informatics, Volume 75, Issue 6, June 2006, pp. 430-442</div></div><div class="buttons"><a href="http://www.sciencedirect.com/science/article/pii/S1386505605000912/pdfft?md5=cfedbddb864b8b22b92ed35c1cf3ec7c&amp;pid=1-s2.0-S1386505605000912-main.pdf" class="icon-link" rel="nofollow" aria-describedby="special-issue-articles-article0-title" target="_blank" data-hack="#"><i class="icon-pdf"></i><span class="primary size-m">Download PDF</span></a><a class="icon-link arrow-btn" href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#" aria-describedby="special-issue-articles-article0-title" aria-controls="special-issue-articles-article0" aria-expanded="false"><span class="Icon ArrowDown" style="display: inline-block; height: 12px; width: 12px; margin-right: 8px; transition: transform 0.3s, -webkit-transform 0.3s;"><svg fill="currentColor" tabindex="-1" focusable="false" height="16" width="16" viewBox="0 0 16 16" style="width: 100%; height: 100%;"><path d="M16 4.452l-1.26-1.26L8 9.932l-6.74-6.74L0 4.452l8 8 8-8z"></path></svg></span><span class="view-more">View details</span></a></div><div id="special-issue-articles-article0" aria-hidden="true"></div></li><li class="SidePanelItem"><div class="sub-heading"><a href="http://www.sciencedirect.com/science/article/pii/S1386505605000924" data-hack="#"><h3 class="article-title ellipsis" id="special-issue-articles-article1-title"><span>A hybrid method for relation extraction from biomedical literature</span></h3></a><div class="article-source ellipsis">International Journal of Medical Informatics, Volume 75, Issue 6, June 2006, pp. 443-455</div></div><div class="buttons"><a href="http://www.sciencedirect.com/science/article/pii/S1386505605000924/pdfft?md5=350f28c46454d062d25e4470ccb51e96&amp;pid=1-s2.0-S1386505605000924-main.pdf" class="icon-link" rel="nofollow" aria-describedby="special-issue-articles-article1-title" target="_blank" data-hack="#"><i class="icon-pdf"></i><span class="primary size-m">Download PDF</span></a><a class="icon-link arrow-btn" href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#" aria-describedby="special-issue-articles-article1-title" aria-controls="special-issue-articles-article1" aria-expanded="false"><span class="Icon ArrowDown" style="display: inline-block; height: 12px; width: 12px; margin-right: 8px; transition: transform 0.3s, -webkit-transform 0.3s;"><svg fill="currentColor" tabindex="-1" focusable="false" height="16" width="16" viewBox="0 0 16 16" style="width: 100%; height: 100%;"><path d="M16 4.452l-1.26-1.26L8 9.932l-6.74-6.74L0 4.452l8 8 8-8z"></path></svg></span><span class="view-more">View details</span></a></div><div id="special-issue-articles-article1" aria-hidden="true"></div></li><li class="SidePanelItem"><div class="sub-heading"><a href="http://www.sciencedirect.com/science/article/pii/S1386505605001218" data-hack="#"><h3 class="article-title ellipsis" id="special-issue-articles-article2-title"><span>Recognizing names in biomedical texts using mutual information independence model and SVM plus sigmoid</span></h3></a><div class="article-source ellipsis">International Journal of Medical Informatics, Volume 75, Issue 6, June 2006, pp. 456-467</div></div><div class="buttons"><a href="http://www.sciencedirect.com/science/article/pii/S1386505605001218/pdfft?md5=aca6cd38c7fc6b5c100ac8cfa1c5775b&amp;pid=1-s2.0-S1386505605001218-main.pdf" class="icon-link" rel="nofollow" aria-describedby="special-issue-articles-article2-title" target="_blank" data-hack="#"><i class="icon-pdf"></i><span class="primary size-m">Download PDF</span></a><a class="icon-link arrow-btn" href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#" aria-describedby="special-issue-articles-article2-title" aria-controls="special-issue-articles-article2" aria-expanded="false"><span class="Icon ArrowDown" style="display: inline-block; height: 12px; width: 12px; margin-right: 8px; transition: transform 0.3s, -webkit-transform 0.3s;"><svg fill="currentColor" tabindex="-1" focusable="false" height="16" width="16" viewBox="0 0 16 16" style="width: 100%; height: 100%;"><path d="M16 4.452l-1.26-1.26L8 9.932l-6.74-6.74L0 4.452l8 8 8-8z"></path></svg></span><span class="view-more">View details</span></a></div><div id="special-issue-articles-article2" aria-hidden="true"></div></li></ul><a href="http://www.sciencedirect.com/science/journal/13865056/75/6" class="view-more-link" target="_blank" aria-describedby="special-issue-articles-header">View more articles<span class="Icon ArrowDown" style="display: inline-block; height: 8px; width: 8px;"><svg fill="currentColor" tabindex="-1" focusable="false" height="16" width="16" viewBox="0 0 16 16" style="width: 100%; height: 100%;"><path d="M16 4.452l-1.26-1.26L8 9.932l-6.74-6.74L0 4.452l8 8 8-8z"></path></svg></span></a></div><div></div></section></section><section class="SidePanel"><header class="side-panel-header" id="recommended-articles-header"><a class="arrow-btn head" href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#" aria-controls="recommended-articles" aria-expanded="false"><span class="Icon ArrowDown" style="display: inline-block; height: 16px; width: 16px; margin-right: 8px; transition: transform 0.3s, -webkit-transform 0.3s;"><svg fill="currentColor" tabindex="-1" focusable="false" height="16" width="16" viewBox="0 0 16 16" style="width: 100%; height: 100%;"><path d="M16 4.452l-1.26-1.26L8 9.932l-6.74-6.74L0 4.452l8 8 8-8z"></path></svg></span><h2><span class="section-title">Recommended articles</span></h2></a></header><div id="recommended-articles" aria-hidden="true"></div><div style="display:none"></div></section><section class="SidePanel"><header class="side-panel-header" id="citing-articles-header"><a class="arrow-btn head" href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#" aria-controls="citing-articles" aria-expanded="false"><span class="Icon ArrowDown" style="display: inline-block; height: 16px; width: 16px; margin-right: 8px; transition: transform 0.3s, -webkit-transform 0.3s;"><svg fill="currentColor" tabindex="-1" focusable="false" height="16" width="16" viewBox="0 0 16 16" style="width: 100%; height: 100%;"><path d="M16 4.452l-1.26-1.26L8 9.932l-6.74-6.74L0 4.452l8 8 8-8z"></path></svg></span><h2><span class="section-title">Citing articles (23)</span></h2></a></header><div id="citing-articles" aria-hidden="true"></div><div style="display:none"></div></section><section class="SidePanel"><header class="side-panel-header" id="metrics-header"><a class="arrow-btn head rotated" href="http://www.sciencedirect.com/science/article/pii/S1386505605001723?via%3Dihub#" aria-controls="metrics" aria-expanded="true"><span class="Icon ArrowDown" style="display:inline-block;height:16px;width:16px;margin-right:8px;transition:transform .3s, -webkit-transform .3s;transform:scaleY(-1)"><svg fill="currentColor" style="width:100%;height:100%" tabindex="-1" focusable="false" height="16" width="16" viewBox="0 0 16 16"><path d="M16 4.452l-1.26-1.26L8 9.932l-6.74-6.74L0 4.452l8 8 8-8z"></path></svg></span><h2><span class="section-title">Article Metrics</span></h2></a></header><div id="metrics" aria-hidden="true"></div><div><div class="plum-sciencedirect-theme"><div class="PlumX-Summary"><div class="pps-container pps-container-vertical plx-no-print"><div class="pps-branding pps-branding-top"><img src="./Developing a corpus of clinical notes manually annotated for part-of-speech - ScienceDirect_files/plumx-inverse-logo.png" class="plx-logo"></div><div class="pps-cols"><div class="pps-col plx-capture"><div class="plx-capture"><div class="pps-title">Captures</div><ul><li class="plx-capture"><span class="pps-label">Readers: </span><span class="pps-count">41</span></li><li class="plx-capture"><span class="pps-label">Exports-Saves: </span><span class="pps-count">6</span></li></ul></div></div><div class="pps-col plx-citation"><div class="plx-citation"><div class="pps-title">Citations</div><ul><li class="plx-citation"><span class="pps-label">Citation Indexes: </span><span class="pps-count">23</span></li></ul></div></div></div><div><div class="pps-branding pps-branding-bottom"><img src="./Developing a corpus of clinical notes manually annotated for part-of-speech - ScienceDirect_files/plumx-logo.png" class="plx-logo"></div><a target="_blank" href="https://plu.mx/plum/a/?doi=10.1016/j.ijmedinf.2005.08.006&amp;theme=plum-sciencedirect-theme&amp;hideUsage=true" class="pps-seemore" title="PlumX Metrics Detail Page">View details<svg fill="currentColor" tabindex="-1" focusable="false" width="16" height="16" viewBox="0 0 16 16" class="svg-arrow"><path d="M16 4.452l-1.26-1.26L8 9.932l-6.74-6.74L0 4.452l8 8 8-8z"></path></svg></a></div></div></div></div></div></section></aside></div></div><div></div></div></section></div></div></div></div></div>
<script type="application/json" data-iso-key="_0">{"config":{"reduxLogging":false,"assetsBaseUrl":"https://sdfestaticassets-us-west-2.sciencedirectassets.com/prod/a092afc25f42bb95ff609db917e5198de5d5746f","tokens":{"optimizely":204774041,"usabilla":"eb1c14a91932"},"mediaBaseUrl":"https://ars.els-cdn.com/content/image/","canonicalUrl":"https://www.sciencedirect.com/science/article/pii/S1386505605001723","publicEndpoints":{"pdf":"/sdfe/pdf/download","atp":"/user/api/export"},"googleMapsApiKey":"AIzaSyAV0ZB1XWtRlzJ06pJAW74Oan87lQVgBHY"},"userAgent":"Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Ubuntu Chromium/63.0.3239.84 Chrome/63.0.3239.84 Safari/537.36","abstracts":{"content":[{"#name":"abstract","$":{"xmlns:ce":true,"class":"author","id":"aep-abstract-id15"},"$$":[{"#name":"section-title","_":"Abstract"},{"#name":"abstract-sec","$":{"id":"aep-abstract-sec-id16"},"$$":[{"#name":"section-title","_":"Purpose"},{"#name":"simple-para","$":{"view":"all"},"_":"This paper presents a project whose main goal is to construct a corpus of clinical text manually annotated for part-of-speech (POS) information. We describe and discuss the process of training three domain experts to perform linguistic annotation."}]},{"#name":"abstract-sec","$":{"id":"aep-abstract-sec-id17"},"$$":[{"#name":"section-title","_":"Methods"},{"#name":"simple-para","$":{"view":"all"},"_":"Three domain experts were trained to perform manual annotation of a corpus of clinical notes. A part of this corpus was combined with the Penn Treebank corpus of general purpose English text and another part was set aside for testing. The corpora were then used for training and testing statistical part-of-speech taggers. We list some of the challenges as well as encouraging results pertaining to inter-rater agreement and consistency of annotation."}]},{"#name":"abstract-sec","$":{"id":"aep-abstract-sec-id18"},"$$":[{"#name":"section-title","_":"Results"},{"#name":"simple-para","$":{"view":"all"},"_":"We used the Trigrams‘n’Tags (TnT) [T. Brants, TnT—a statistical part-of-speech tagger, In: Proceedings of NAACL/ANLP-2000 Symposium, 2000] tagger trained on general English data to achieve 89.79% correctness. The same tagger trained on a portion of the medical data annotated for this project improved the performance to 94.69%. Furthermore, we find that discriminating between different types of discourse represented by different sections of clinical text may be very beneficial to improve correctness of POS tagging."}]},{"#name":"abstract-sec","$":{"id":"aep-abstract-sec-id19"},"$$":[{"#name":"section-title","_":"Conclusion"},{"#name":"simple-para","$":{"view":"all"},"_":"Our preliminary experimental results indicate the necessity for adapting state-of-the-art POS taggers to the sublanguage domain of clinical text."}]}]}],"floats":[],"footnotes":[],"attachments":[]},"biographies":{},"combinedContentItems":{"content":[{"#name":"keywords","$$":[{"#name":"keywords","$":{"xmlns:ce":true,"xmlns:aep":true,"xmlns:xoe":true,"xmlns:mml":true,"xmlns:xs":true,"xmlns:xlink":true,"xmlns:xocs":true,"xmlns:tb":true,"xmlns:xsi":true,"xmlns:cals":true,"xmlns:sb":true,"xmlns:ja":true,"xmlns":true,"class":"keyword","id":"aep-keywords-id20"},"$$":[{"#name":"section-title","_":"Keywords"},{"#name":"keyword","$$":[{"#name":"text","_":"Natural language processing"}]},{"#name":"keyword","$$":[{"#name":"text","_":"Statistical part-of-speech tagging"}]},{"#name":"keyword","$$":[{"#name":"text","_":"Domain adaptation"}]},{"#name":"keyword","$$":[{"#name":"text","_":"Medical domain"}]},{"#name":"keyword","$$":[{"#name":"text","_":"Text analysis"}]},{"#name":"keyword","$$":[{"#name":"text","_":"Manual text annotation"}]}]}]},{"#name":"title-editors-groups","$":{"xmlns:xocs":true,"xmlns:xoe":true,"xmlns:mml":true,"xmlns:ce":true,"xmlns:xs":true,"xmlns:xlink":true,"xmlns:tb":true,"xmlns:xsi":true,"xmlns:cals":true,"xmlns:sb":true,"xmlns:ja":true,"xmlns":true},"$$":[{"#name":"title","_":"Recent Advances in Natural Language Processing for Biomedical Applications Special Issue"},{"#name":"editors","$$":[{"#name":"author-group","$$":[{"#name":"author","$":{"id":"aep-author-id1"},"$$":[{"#name":"given-name","_":"Robert"},{"#name":"surname","_":"Baud"}]},{"#name":"affiliation","$":{"id":"aep-affiliation-id2"},"$$":[{"#name":"textfn","_":"Medical Informatics and Radiology Department University Hospital of Geneva24 Micheli du Crest1200 GeneveSwitzerland"}]}]},{"#name":"author-group","$$":[{"#name":"author","$":{"id":"aep-author-id3"},"$$":[{"#name":"given-name","_":"Nigel"},{"#name":"surname","_":"Collier"}]},{"#name":"affiliation","$":{"id":"aep-affiliation-id4"},"$$":[{"#name":"textfn","_":"Associate ProfessorNational Institute of Informatics2-1-2 Hitotsubashi, Chiyoda-kuTokyo 101-8430Japan "}]}]},{"#name":"author-group","$$":[{"#name":"author","$":{"id":"aep-author-id5"},"$$":[{"#name":"given-name","_":"Adeline"},{"#name":"surname","_":"Nazarenko"}]},{"#name":"affiliation","$":{"id":"aep-affiliation-id6"},"$$":[{"#name":"textfn","_":"Laboratoire d'Informatique de Paris-NordCNRS & Universite Paris-Nord 99, av. J-B Clement 93430 Villetaneuse France"}]}]},{"#name":"author-group","$$":[{"#name":"author","$":{"id":"aep-author-id7"},"$$":[{"#name":"given-name","_":"Patrick"},{"#name":"surname","_":"Ruch"}]},{"#name":"affiliation","$":{"id":"aep-affiliation-id8"},"$$":[{"#name":"textfn","_":"Medical Informatics and Radiology Department University and University Hospital of Geneva1 General Dufour1201 GeneveSwitzerland"}]}]}]}]}],"floats":[],"footnotes":[],"attachments":[]},"experiments":{},"rawtext":"","authors":{"content":[{"#name":"author-group","$":{"xmlns:ce":true},"$$":[{"#name":"author","$":{"id":"aep-author-id12"},"$$":[{"#name":"given-name","_":"Serguei V."},{"#name":"surname","_":"Pakhomov"},{"#name":"cross-ref","$":{"refid":"aff1"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"a"}]},{"#name":"cross-ref","$":{"refid":"cor1"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"⁎"}]},{"#name":"e-address","$":{"type":"email"},"_":"Pakhomov.Serguei@mayo.edu"}]},{"#name":"author","$":{"id":"aep-author-id13"},"$$":[{"#name":"given-name","_":"Anni"},{"#name":"surname","_":"Coden"},{"#name":"cross-ref","$":{"refid":"aff2"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"b"}]},{"#name":"e-address","$":{"type":"email"},"_":"anni@us.ibm.com"}]},{"#name":"author","$":{"id":"aep-author-id14"},"$$":[{"#name":"given-name","_":"Christopher G."},{"#name":"surname","_":"Chute"},{"#name":"cross-ref","$":{"refid":"aff1"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"a"}]},{"#name":"e-address","$":{"type":"email"},"_":"Chute@mayo.edu"}]},{"#name":"affiliation","$":{"id":"aff1"},"$$":[{"#name":"label","_":"a"},{"#name":"textfn","_":"Division of Biomedical Informatics, Mayo College of Medicine, Rochester, MN 55905, USA"}]},{"#name":"affiliation","$":{"id":"aff2"},"$$":[{"#name":"label","_":"b"},{"#name":"textfn","_":"IBM, T.J. Watson Research Center, Hawthorne, NY 10532, USA"}]},{"#name":"correspondence","$":{"id":"cor1"},"$$":[{"#name":"label","_":"⁎"},{"#name":"text","_":"Corresponding author. Tel.: +1 507 538 1774."}]}]}],"floats":[],"footnotes":[],"affiliations":{"aff1":{"#name":"affiliation","$":{"id":"aff1"},"$$":[{"#name":"label","_":"a"},{"#name":"textfn","_":"Division of Biomedical Informatics, Mayo College of Medicine, Rochester, MN 55905, USA"}]},"aff2":{"#name":"affiliation","$":{"id":"aff2"},"$$":[{"#name":"label","_":"b"},{"#name":"textfn","_":"IBM, T.J. Watson Research Center, Hawthorne, NY 10532, USA"}]}},"correspondences":{"cor1":{"#name":"correspondence","$":{"id":"cor1"},"$$":[{"#name":"label","_":"⁎"},{"#name":"text","_":"Corresponding author. Tel.: +1 507 538 1774."}]}},"attachments":[],"scopusAuthorIds":{},"articles":{}},"body":{},"exam":{},"article":{"publication-content":{"noElsevierLogo":false,"imprintPublisher":{"displayName":"Elsevier","id":"47"},"isSpecialIssue":true,"altSmallCover":"","isSampleIssue":false,"transactionsBlocked":false,"publicationOpenAccess":{"oaStatus":"","oaArticleCount":39,"openArchiveStatus":false,"openArchiveArticleCount":0,"openAccessStartDate":"","oaAllowsAuthorPaid":true},"titleImageUrl":"","issue-cover":{"attachment":[{"attachment-eid":"1-s2.0-S1386505606X00722-cov200h.gif","file-basename":"cov200h","extension":"gif","filename":"cov200h.gif","ucs-locator":["https://s3.amazonaws.com/prod-ucs-content-store-us-east/content/pii:S1386505606X00722/loc/DOWNSAMPLED200/image/gif/b380d4c1f95b569aae25f914ac506011/cov200h.gif","https://s3-eu-west-1.amazonaws.com/prod-ucs-content-store-eu-west/content/pii:S1386505606X00722/loc/DOWNSAMPLED200/image/gif/b380d4c1f95b569aae25f914ac506011/cov200h.gif"],"attachment-type":"IMAGE-COVER-H200","filesize":"17866","pixel-height":"200","pixel-width":"150"},{"attachment-eid":"1-s2.0-S1386505606X00722-cov150h.gif","file-basename":"cov150h","extension":"gif","filename":"cov150h.gif","ucs-locator":["https://s3.amazonaws.com/prod-ucs-content-store-us-east/content/pii:S1386505606X00722/loc/DOWNSAMPLED/image/gif/af5ba8b8d50c835ec3fb0058ebada635/cov150h.gif","https://s3-eu-west-1.amazonaws.com/prod-ucs-content-store-eu-west/content/pii:S1386505606X00722/loc/DOWNSAMPLED/image/gif/af5ba8b8d50c835ec3fb0058ebada635/cov150h.gif"],"attachment-type":"IMAGE-COVER-H150","filesize":"11122","pixel-height":"150","pixel-width":"112"}]},"smallCoverUrl":"https://ars.els-cdn.com/content/image/S13865056.gif","serialCoverPgUrl":"","sourceOpenAccess":false,"coverImageUrl":"https://ars.els-cdn.com/content/image/1-s2.0-S1386505606X00722-cov150h.gif"},"pii":"S1386505605001723","dates":{"Received":"6 April 2005","Revised":["23 July 2005"],"Accepted":"10 August 2005","Publication date":"1 June 2006","Available online":"19 September 2005"},"access":{},"crawlerInformation":{"canCrawlPDFContent":false,"isCrawler":false},"cid":"271161","content-family":"serial","copyright-line":"Copyright © 2005 Elsevier Ireland Ltd. All rights reserved.","cover-date-years":["2006"],"cover-date-start":"2006-06-01","cover-date-text":"June 2006","document-subtype":"fla","document-type":"article","eid":"1-s2.0-S1386505605001723","first-fp":"418","hub-eid":"1-s2.0-S1386505606X00722","iss-first":"6","item-weight":"FULL-TEXT","language":"en","last-lp":"429","last-author":{"#name":"last-author","$":{"xmlns:dm":true},"$$":[{"#name":"author","$":{"xmlns:ce":true,"id":"aep-author-id14"},"$$":[{"#name":"given-name","_":"Christopher G."},{"#name":"surname","_":"Chute"},{"#name":"cross-ref","$":{"refid":"aff1"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"a"}]},{"#name":"e-address","$":{"type":"email"},"_":"Chute@mayo.edu"}]}]},"srctitle":"International Journal of Medical Informatics","vol-first":"75","vol-iss-suppl-text":"Volume 75, Issue 6","doi":"10.1016/j.ijmedinf.2005.08.006","userSettings":{"forceAbstract":false},"pages":[{"last-page":"429","first-page":"418"}],"normalized-first-auth-initial":"S","normalized-first-auth-surname":"PAKHOMOV","entitledToken":"167CC834BDC1E1A5B8C060D8C058EE9B2D3EABDFA91E06A1C37D615F327903917C5DA112CA4DA958","timestamp":"2010-10-09T23:16:03.158889-04:00","issuePii":"S1386505606X00722","title":{"content":[{"#name":"title","$":{"xmlns:ce":true},"_":"Developing a corpus of clinical notes manually annotated for part-of-speech"}],"floats":[],"footnotes":[],"attachments":[]},"contentType":"JL","crossmark":false,"issn":"13865056","issnFormatted":"1386-5056","useEnhancedReader":false,"isCorpReq":false,"pdfDownload":{"linkType":"DOWNLOAD","linkToPdf":"/science/article/pii/S1386505605001723/pdfft?md5=566710269dc6c26937e3e391a2dd14ff&pid=1-s2.0-S1386505605001723-main.pdf","isPdfFullText":false,"fileName":"1-s2.0-S1386505605001723-main.pdf"},"pdfUrlForCrawlers":"https://www.sciencedirect.com/science/article/pii/S1386505605001723/pdfft?md5=566710269dc6c26937e3e391a2dd14ff&pid=1-s2.0-S1386505605001723-main.pdf","indexTag":true,"volRange":"75","issRange":"6","ssoUrls":["//acw.scopus.com/SSOCore/update?acw=f6b11ab3357120440d9bedb39e20e703acb7gxrqa%7C%24%7C489DADFEBC18A6DE3AB2A7AFC5E590E11266774EDC4E95AE1BECF85F55B54749E96C6CA04FDAF5D487526B853A05B72A575D927D474BD41A3FBA44D1BD4E4F2EAFE9C31A29ED2080B6DA1F7CB1786ABB&utt=a1ba400897e0613c2a2cd5c52a365d6e8e55c2","//acw.sciencedirect.com/SSOCore/update?acw=f6b11ab3357120440d9bedb39e20e703acb7gxrqa%7C%24%7C489DADFEBC18A6DE3AB2A7AFC5E590E11266774EDC4E95AE1BECF85F55B54749E96C6CA04FDAF5D487526B853A05B72A575D927D474BD41A3FBA44D1BD4E4F2EAFE9C31A29ED2080B6DA1F7CB1786ABB&utt=a1ba400897e0613c2a2cd5c52a365d6e8e55c2","//acw.mendeley.com/SSOCore/update?acw=f6b11ab3357120440d9bedb39e20e703acb7gxrqa%7C%24%7C489DADFEBC18A6DE3AB2A7AFC5E590E11266774EDC4E95AE1BECF85F55B54749E96C6CA04FDAF5D487526B853A05B72A575D927D474BD41A3FBA44D1BD4E4F2EAFE9C31A29ED2080B6DA1F7CB1786ABB&utt=a1ba400897e0613c2a2cd5c52a365d6e8e55c2","//acw.sciverse.com/SSOCore/update?acw=f6b11ab3357120440d9bedb39e20e703acb7gxrqa%7C%24%7C489DADFEBC18A6DE3AB2A7AFC5E590E11266774EDC4E95AE1BECF85F55B54749E96C6CA04FDAF5D487526B853A05B72A575D927D474BD41A3FBA44D1BD4E4F2EAFE9C31A29ED2080B6DA1F7CB1786ABB&utt=a1ba400897e0613c2a2cd5c52a365d6e8e55c2","//acw.evise.com/SSOCore/update?acw=f6b11ab3357120440d9bedb39e20e703acb7gxrqa%7C%24%7C489DADFEBC18A6DE3AB2A7AFC5E590E11266774EDC4E95AE1BECF85F55B54749E96C6CA04FDAF5D487526B853A05B72A575D927D474BD41A3FBA44D1BD4E4F2EAFE9C31A29ED2080B6DA1F7CB1786ABB&utt=a1ba400897e0613c2a2cd5c52a365d6e8e55c2","//acw.elsevier.com/SSOCore/update?acw=f6b11ab3357120440d9bedb39e20e703acb7gxrqa%7C%24%7C489DADFEBC18A6DE3AB2A7AFC5E590E11266774EDC4E95AE1BECF85F55B54749E96C6CA04FDAF5D487526B853A05B72A575D927D474BD41A3FBA44D1BD4E4F2EAFE9C31A29ED2080B6DA1F7CB1786ABB&utt=a1ba400897e0613c2a2cd5c52a365d6e8e55c2"],"userProfile":{"accountId":"50229","departmentId":"71116"},"entitlementReason":"package","articleEntitlement":{"entitled":true},"aipType":"none","downloadFullIssue":true,"open-research":{},"titleString":"Developing a corpus of clinical notes manually annotated for part-of-speech","isAbstract":false,"onAbstractWhitelist":false,"isContentVisible":false,"ajaxLinks":{"citingArticles":true,"references":true,"referredToBy":true,"toc":true,"specialIssueArticles":true,"body":true,"recommendations":true},"userId":"1007916"},"specialIssueArticles":{},"recommendations":{},"entitledRecommendations":{"isOpen":false,"articles":[],"selected":[]},"citingArticles":{},"workspace":{"isOpen":false},"authorProfile":{"isOpen":false},"crossMark":{"isOpen":false},"userIdentity":{},"refersTo":{},"referredToBy":{},"downloadIssue":{"isOpen":false,"articles":[],"selected":[]},"references":{},"referenceLinks":{"internal":{},"external":{}},"glossary":{},"relatedContent":{"isModal":false,"isOpenSpecialIssueArticles":true,"isOpenRecommendations":false,"isOpenCitingArticles":false,"citingArticles":[false,false,false],"recommendations":[false,false,false],"specialIssueArticles":[false,false,false]},"banner":{"expanded":false},"transientError":{"isOpen":false},"pdfDropdown":{},"exportCitation":{"isOpen":false},"tableOfContents":{"showEntitledTocLinks":true},"enrichedContent":{"tableOfContents":false,"researchData":{"hasResearchData":false,"dataProfile":{},"openData":{},"mendeleyData":{},"databaseLinking":{}},"geospatialData":{"attachments":[]},"interactiveCaseInsights":{}},"metrics":{"isOpen":true},"issueNavigation":{"previous":{},"next":{}}}</script>
      <footer>
        <div class="ccs-component"><div class="Footer" role="contentinfo"><div class="els-footer"><div class="els-footer-body"><a class="els-footer-elsevier" href="https://www.elsevier.com/" rel="nofollow" target="_blank"><span class="ccs-icon svg-wordmark-elsevier" aria-label="Elsevier"><span class="svg-text">Elsevier</span></span></a><div class="els-footer-content"><div class="els-footer-links"><a class="link-inherit" href="https://www.elsevier.com/solutions/sciencedirect" rel="nofollow" target="_blank">About ScienceDirect</a><wbr><a class="link-inherit" href="http://www.sciencedirect.com/science/activateaccess" rel="nofollow" target="_blank">Remote access</a><wbr><a class="link-inherit" href="http://www.sciencedirect.com/science?_ob=ShoppingCartURL&amp;_method=display&amp;md5=3ff44acb300f01481824c54a2973d019" rel="nofollow" target="_blank">Shopping cart</a><wbr><a class="link-inherit" href="https://service.elsevier.com/app/contact/supporthub/sciencedirect/" rel="nofollow" target="_blank">Contact and support</a><wbr><a class="link-inherit" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions" rel="nofollow" target="_blank">Terms and conditions</a><wbr><a class="link-inherit" href="https://www.elsevier.com/legal/privacy-policy" rel="nofollow" target="_blank">Privacy policy</a></div><div class="els-footer-cookie">Cookies are used by this site. For more information, visit the <a class="link-blue" href="https://www.elsevier.com/solutions/sciencedirect/support/cookies" rel="nofollow" target="_blank">cookies page</a>.</div><div class="els-footer-copyright">Copyright © <span class="copyright-year">2018</span> Elsevier B.V. or its licensors or contributors. ScienceDirect ® is a registered trademark of Elsevier B.V.</div></div></div><div class="els-footer-relx-group"><a class="els-footer-relx" href="https://www.relx.com/" rel="nofollow" target="_blank"><span class="ccs-icon svg-wordmark-relx-group" aria-label="RELX Group"><span class="svg-text">RELX Group</span></span></a></div></div><script>if(typeof fbq === 'undefined') {
    !function(f,b,e,v,n,t,s){if(f.fbq)return;
    n=f.fbq=function(){n.callMethod?n.callMethod.apply(n,arguments):n.queue.push(arguments)};
    if(!f._fbq)f._fbq=n;
    n.push=n;
    n.loaded=!0;n.version='2.0';n.queue=[];t=b.createElement(e);
    t.async=!0;t.src=v;s=b.getElementsByTagName(e)[0];
    s.parentNode.insertBefore(t,s)}(window,document,'script','https://connect.facebook.net/en_US/fbevents.js');
    fbq('set', 'autoConfig', 'false', '1086177924859222') // disable auto tracking of EVERYTHING
    fbq('init', '1086177924859222'); // Insert your pixel ID here.
 }
 fbq('track', 'PageView');</script><noscript>&lt;img height="1" width="1" style="display:none;" src="https://www.facebook.com/tr?id=1086177924859222&amp;amp;ev=PageView&amp;amp;noscript=1"/&gt;</noscript><script type="text/javascript">/* <![CDATA[ */
                        var google_conversion_id = 1017027555;
                        var google_custom_params = window.google_tag_params;
                        var google_remarketing_only = true;
                        /* ]]> */</script><script type="text/javascript" src="./Developing a corpus of clinical notes manually annotated for part-of-speech - ScienceDirect_files/conversion.js"></script><noscript>&lt;div style="display:inline;"&gt;&lt;img height="1" width="1" style="border_style:none;" alt="" src="//googleads.g.doubleclick.net/pagead/viewthroughconversion/1017027555/?guid=ON&amp;amp;script=0"/&gt;&lt;/div&gt;</noscript></div></div>
      </footer>
      <iframe style="display: none" src="./Developing a corpus of clinical notes manually annotated for part-of-speech - ScienceDirect_files/update.html" tabindex="-1"></iframe><iframe style="display: none" src="./Developing a corpus of clinical notes manually annotated for part-of-speech - ScienceDirect_files/update(1).html" tabindex="-1"></iframe><iframe style="display: none" src="./Developing a corpus of clinical notes manually annotated for part-of-speech - ScienceDirect_files/update(2).html" tabindex="-1"></iframe><iframe style="display: none" src="./Developing a corpus of clinical notes manually annotated for part-of-speech - ScienceDirect_files/update(3).html" tabindex="-1"></iframe><iframe style="display: none" src="./Developing a corpus of clinical notes manually annotated for part-of-speech - ScienceDirect_files/update(4).html" tabindex="-1"></iframe><iframe style="display: none" src="./Developing a corpus of clinical notes manually annotated for part-of-speech - ScienceDirect_files/update(5).html" tabindex="-1"></iframe>
      <script src="./Developing a corpus of clinical notes manually annotated for part-of-speech - ScienceDirect_files/client.js"></script>
      <script src="./Developing a corpus of clinical notes manually annotated for part-of-speech - ScienceDirect_files/satelliteLib-b7cfe8df39a4e5eec5536bba80e13f4b6fa0dd7c.js"></script><script src="./Developing a corpus of clinical notes manually annotated for part-of-speech - ScienceDirect_files/satellite-565e008964746d4385002642.js"></script><iframe sandbox="allow-scripts allow-same-origin" title="Adobe ID Syncing iFrame" id="destination_publishing_iframe_elsevier_0" src="./Developing a corpus of clinical notes manually annotated for part-of-speech - ScienceDirect_files/dest5.html" style="display: none; width: 0px; height: 0px;" class="aamIframeLoaded"></iframe>
      <script src="./Developing a corpus of clinical notes manually annotated for part-of-speech - ScienceDirect_files/react.production.min.js"></script>
      <script src="./Developing a corpus of clinical notes manually annotated for part-of-speech - ScienceDirect_files/react-dom.production.min.js"></script>
      <script src="./Developing a corpus of clinical notes manually annotated for part-of-speech - ScienceDirect_files/arp.js" async=""></script>
      <!-- begin usabilla live embed code -->
      <script type="text/javascript">
        window.lightningjs||function(c){function g(b,d){d&&(d+=(/\?/.test(d)?"&":"?")+"lv=1");c[b]||function(){var i=window,h=document,j=b,g=h.location.protocol,l="load",k=0;(function(){function b(){a.P(l);a.w=1;c[j]("_load")}c[j]=function(){function m(){m.id=e;return c[j].apply(m,arguments)}var b,e=++k;b=this&&this!=i?this.id||0:0;(a.s=a.s||[]).push([e,b,arguments]);m.then=function(b,c,h){var d=a.fh[e]=a.fh[e]||[],j=a.eh[e]=a.eh[e]||[],f=a.ph[e]=a.ph[e]||[];b&&d.push(b);c&&j.push(c);h&&f.push(h);return m};return m};var a=c[j]._={};a.fh={};a.eh={};a.ph={};a.l=d?d.replace(/^\/\//,(g=="https:"?g:"http:")+"//"):d;a.p={0:+new Date};a.P=function(b){a.p[b]=new Date-a.p[0]};a.w&&b();i.addEventListener?i.addEventListener(l,b,!1):i.attachEvent("on"+l,b);var q=function(){function b(){return["<head></head><",c,' onload="var d=',n,";d.getElementsByTagName('head')[0].",d,"(d.",g,"('script')).",i,"='",a.l,"'\"></",c,">"].join("")}var c="body",e=h[c];if(!e)return setTimeout(q,100);a.P(1);var d="appendChild",g="createElement",i="src",k=h[g]("div"),l=k[d](h[g]("div")),f=h[g]("iframe"),n="document",p;k.style.display="none";e.insertBefore(k,e.firstChild).id=o+"-"+j;f.frameBorder="0";f.id=o+"-frame-"+j;/MSIE[ ]+6/.test(navigator.userAgent)&&(f[i]="javascript:false");f.allowTransparency="true";l[d](f);try{f.contentWindow[n].open()}catch(s){a.domain=h.domain,p="javascript:var d="+n+".open();d.domain='"+h.domain+"';",f[i]=p+"void(0);"}try{var r=f.contentWindow[n];r.write(b());r.close()}catch(t){f[i]=p+'d.write("'+b().replace(/"/g,String.fromCharCode(92)+'"')+'");d.close();'}a.P(2)};a.l&&setTimeout(q,0)})()}();c[b].lv="1";return c[b]}var o="lightningjs",k=window[o]=g(o);k.require=g;k.modules=c}({});
        window.usabilla_live = lightningjs.require("usabilla_live", "https://w.usabilla.com/eb1c14a91932.js");
        var customData = {};

        if(window.pageData && pageData.content && pageData.content[0]) {
          customData.entitlementType = pageData.content[0].entitlementType;
        }
        if(window.pageData && pageData.visitor) {
          customData.accessType = pageData.visitor.accessType;
          customData.accountId = pageData.visitor.accountId;
          customData.loginStatus = pageData.visitor.loginStatus;
        }
        usabilla_live("data", {"custom": customData });
      </script>
      <!-- end usabilla live embed code -->
      <script type="text/x-mathjax-config;executed=true">
        MathJax.Hub.Config({
          displayAlign: 'left',
          "fast-preview": {
            disabled: true
          },
          CommonHTML: { linebreaks: { automatic: true } },
          PreviewHTML: { linebreaks: { automatic: true } },
          'HTML-CSS': { linebreaks: { automatic: true } },
          SVG: {
            scale: 90,
            linebreaks: { automatic: true }
          }
        });
      </script>
      <script async="" src="./Developing a corpus of clinical notes manually annotated for part-of-speech - ScienceDirect_files/MathJax.js"></script>
      <script async="" src="./Developing a corpus of clinical notes manually annotated for part-of-speech - ScienceDirect_files/gpt.js"></script>
    
  <div class="adsbox">&nbsp;</div><div class="ReactModalPortal"></div><div class="ReactModalPortal"></div><div class="ReactModalPortal"></div><div class="ReactModalPortal"></div><script src="./Developing a corpus of clinical notes manually annotated for part-of-speech - ScienceDirect_files/widget-summary.js" async=""></script><div class="usabilla_live_button_container" tabindex="0" style="position:fixed;width:117px;height:34px;z-index:600;bottom:0px;right:2%" aria-label="Usabilla Feedback Button"><iframe src="./Developing a corpus of clinical notes manually annotated for part-of-speech - ScienceDirect_files/saved_resource(2).html" frameborder="0" marginwidth="0" marginheight="0" scrolling="no" data-tags="bottom" title="Usabilla Feedback Button" style="width: 117px; height: 34px; margin: 0px; padding: 0px; border: 0px; overflow: hidden; z-index: 9998; position: absolute; left: 0px; top: 0px; box-shadow: 0px 0px 0px; background-color: transparent;"></iframe></div><div class="usabilla__overlay" style="position: fixed; background-color: transparent; z-index: 2147483647; visibility: visible; margin: 0px; padding: 0px; overflow: hidden; text-align: left; right: 0px; top: 395px; width: 540px; height: 148px;"><iframe frameborder="0" marginwidth="0" marginheight="0" scrolling="no" src="./Developing a corpus of clinical notes manually annotated for part-of-speech - ScienceDirect_files/034668d8b6fb5d900648dbc8bccb4058a4abdf01.html" style="width: 540px; height: 148px; margin: 0px; padding: 0px; border: 0px; overflow: hidden; z-index: 9997; max-width: 100%; position: absolute; background-color: transparent; visibility: visible; box-shadow: 0px 0px 0px; display: block; left: 0px;"></iframe></div></body></html>